{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "source": [
        "## Подготовка hmmer\n"
      ],
      "metadata": {
        "id": "47Ku6MsZKHdn"
      }
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "OL65yS9-IrVm",
        "outputId": "0aaf4e42-8461-479c-ec6b-7744433b9948"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "--2024-06-19 19:22:26--  http://eddylab.org/software/hmmer/hmmer-3.3.2.tar.gz\n",
            "Resolving eddylab.org (eddylab.org)... 96.126.110.11, 2600:3c03::f03c:91ff:fec8:383c\n",
            "Connecting to eddylab.org (eddylab.org)|96.126.110.11|:80... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 18213049 (17M) [application/x-gzip]\n",
            "Saving to: ‘hmmer-3.3.2.tar.gz’\n",
            "\n",
            "     0K .......... .......... .......... .......... ..........  0% 4.13M 4s\n",
            "    50K .......... .......... .......... .......... ..........  0% 7.09M 3s\n",
            "   100K .......... .......... .......... .......... ..........  0%  255M 2s\n",
            "   150K .......... .......... .......... .......... ..........  1% 4.09M 3s\n",
            "   200K .......... .......... .......... .......... ..........  1% 15.8M 2s\n",
            "   250K .......... .......... .......... .......... ..........  1%  198M 2s\n",
            "   300K .......... .......... .......... .......... ..........  1% 8.09M 2s\n",
            "   350K .......... .......... .......... .......... ..........  2% 19.2M 2s\n",
            "   400K .......... .......... .......... .......... ..........  2%  300M 2s\n",
            "   450K .......... .......... .......... .......... ..........  2%  285M 1s\n",
            "   500K .......... .......... .......... .......... ..........  3%  265M 1s\n",
            "   550K .......... .......... .......... .......... ..........  3% 10.8M 1s\n",
            "   600K .......... .......... .......... .......... ..........  3% 26.2M 1s\n",
            "   650K .......... .......... .......... .......... ..........  3% 53.2M 1s\n",
            "   700K .......... .......... .......... .......... ..........  4% 36.7M 1s\n",
            "   750K .......... .......... .......... .......... ..........  4%  202M 1s\n",
            "   800K .......... .......... .......... .......... ..........  4%  166M 1s\n",
            "   850K .......... .......... .......... .......... ..........  5%  251M 1s\n",
            "   900K .......... .......... .......... .......... ..........  5% 44.4M 1s\n",
            "   950K .......... .......... .......... .......... ..........  5%  278M 1s\n",
            "  1000K .......... .......... .......... .......... ..........  5% 25.4M 1s\n",
            "  1050K .......... .......... .......... .......... ..........  6%  271M 1s\n",
            "  1100K .......... .......... .......... .......... ..........  6%  237M 1s\n",
            "  1150K .......... .......... .......... .......... ..........  6%  278M 1s\n",
            "  1200K .......... .......... .......... .......... ..........  7%  273M 1s\n",
            "  1250K .......... .......... .......... .......... ..........  7% 14.3M 1s\n",
            "  1300K .......... .......... .......... .......... ..........  7% 3.88M 1s\n",
            "  1350K .......... .......... .......... .......... ..........  7%  139M 1s\n",
            "  1400K .......... .......... .......... .......... ..........  8%  240M 1s\n",
            "  1450K .......... .......... .......... .......... ..........  8%  287M 1s\n",
            "  1500K .......... .......... .......... .......... ..........  8%  244M 1s\n",
            "  1550K .......... .......... .......... .......... ..........  8%  265M 1s\n",
            "  1600K .......... .......... .......... .......... ..........  9%  294M 1s\n",
            "  1650K .......... .......... .......... .......... ..........  9%  250M 1s\n",
            "  1700K .......... .......... .......... .......... ..........  9%  251M 1s\n",
            "  1750K .......... .......... .......... .......... .......... 10%  291M 1s\n",
            "  1800K .......... .......... .......... .......... .......... 10%  285M 1s\n",
            "  1850K .......... .......... .......... .......... .......... 10% 6.20M 1s\n",
            "  1900K .......... .......... .......... .......... .......... 10% 5.63M 1s\n",
            "  1950K .......... .......... .......... .......... .......... 11% 14.6M 1s\n",
            "  2000K .......... .......... .......... .......... .......... 11% 5.61M 1s\n",
            "  2050K .......... .......... .......... .......... .......... 11% 53.9M 1s\n",
            "  2100K .......... .......... .......... .......... .......... 12% 46.5M 1s\n",
            "  2150K .......... .......... .......... .......... .......... 12% 53.6M 1s\n",
            "  2200K .......... .......... .......... .......... .......... 12% 5.20M 1s\n",
            "  2250K .......... .......... .......... .......... .......... 12%  121M 1s\n",
            "  2300K .......... .......... .......... .......... .......... 13%  231M 1s\n",
            "  2350K .......... .......... .......... .......... .......... 13%  272M 1s\n",
            "  2400K .......... .......... .......... .......... .......... 13%  291M 1s\n",
            "  2450K .......... .......... .......... .......... .......... 14%  304M 1s\n",
            "  2500K .......... .......... .......... .......... .......... 14%  220M 1s\n",
            "  2550K .......... .......... .......... .......... .......... 14%  303M 1s\n",
            "  2600K .......... .......... .......... .......... .......... 14%  262M 1s\n",
            "  2650K .......... .......... .......... .......... .......... 15%  248M 1s\n",
            "  2700K .......... .......... .......... .......... .......... 15%  212M 1s\n",
            "  2750K .......... .......... .......... .......... .......... 15% 7.04M 1s\n",
            "  2800K .......... .......... .......... .......... .......... 16% 56.4M 1s\n",
            "  2850K .......... .......... .......... .......... .......... 16% 16.5M 1s\n",
            "  2900K .......... .......... .......... .......... .......... 16% 9.94M 1s\n",
            "  2950K .......... .......... .......... .......... .......... 16% 53.5M 1s\n",
            "  3000K .......... .......... .......... .......... .......... 17% 16.4M 1s\n",
            "  3050K .......... .......... .......... .......... .......... 17% 53.9M 1s\n",
            "  3100K .......... .......... .......... .......... .......... 17% 47.2M 1s\n",
            "  3150K .......... .......... .......... .......... .......... 17% 14.3M 1s\n",
            "  3200K .......... .......... .......... .......... .......... 18% 9.70M 1s\n",
            "  3250K .......... .......... .......... .......... .......... 18%  245M 1s\n",
            "  3300K .......... .......... .......... .......... .......... 18%  207M 1s\n",
            "  3350K .......... .......... .......... .......... .......... 19%  265M 1s\n",
            "  3400K .......... .......... .......... .......... .......... 19%  301M 1s\n",
            "  3450K .......... .......... .......... .......... .......... 19%  294M 1s\n",
            "  3500K .......... .......... .......... .......... .......... 19%  241M 1s\n",
            "  3550K .......... .......... .......... .......... .......... 20%  256M 1s\n",
            "  3600K .......... .......... .......... .......... .......... 20%  286M 1s\n",
            "  3650K .......... .......... .......... .......... .......... 20%  267M 1s\n",
            "  3700K .......... .......... .......... .......... .......... 21%  228M 1s\n",
            "  3750K .......... .......... .......... .......... .......... 21%  308M 1s\n",
            "  3800K .......... .......... .......... .......... .......... 21% 4.64M 1s\n",
            "  3850K .......... .......... .......... .......... .......... 21% 54.3M 1s\n",
            "  3900K .......... .......... .......... .......... .......... 22% 47.9M 1s\n",
            "  3950K .......... .......... .......... .......... .......... 22% 55.1M 1s\n",
            "  4000K .......... .......... .......... .......... .......... 22% 10.2M 1s\n",
            "  4050K .......... .......... .......... .......... .......... 23% 39.6M 1s\n",
            "  4100K .......... .......... .......... .......... .......... 23% 44.9M 1s\n",
            "  4150K .......... .......... .......... .......... .......... 23% 54.5M 1s\n",
            "  4200K .......... .......... .......... .......... .......... 23% 18.5M 1s\n",
            "  4250K .......... .......... .......... .......... .......... 24% 94.3M 1s\n",
            "  4300K .......... .......... .......... .......... .......... 24%  215M 1s\n",
            "  4350K .......... .......... .......... .......... .......... 24%  262M 1s\n",
            "  4400K .......... .......... .......... .......... .......... 25% 23.8M 1s\n",
            "  4450K .......... .......... .......... .......... .......... 25% 10.2M 1s\n",
            "  4500K .......... .......... .......... .......... .......... 25% 48.5M 1s\n",
            "  4550K .......... .......... .......... .......... .......... 25%  161M 1s\n",
            "  4600K .......... .......... .......... .......... .......... 26%  276M 1s\n",
            "  4650K .......... .......... .......... .......... .......... 26%  294M 1s\n",
            "  4700K .......... .......... .......... .......... .......... 26%  232M 1s\n",
            "  4750K .......... .......... .......... .......... .......... 26%  295M 0s\n",
            "  4800K .......... .......... .......... .......... .......... 27%  276M 0s\n",
            "  4850K .......... .......... .......... .......... .......... 27%  253M 0s\n",
            "  4900K .......... .......... .......... .......... .......... 27%  204M 0s\n",
            "  4950K .......... .......... .......... .......... .......... 28%  244M 0s\n",
            "  5000K .......... .......... .......... .......... .......... 28%  299M 0s\n",
            "  5050K .......... .......... .......... .......... .......... 28%  288M 0s\n",
            "  5100K .......... .......... .......... .......... .......... 28% 10.0M 0s\n",
            "  5150K .......... .......... .......... .......... .......... 29% 55.7M 0s\n",
            "  5200K .......... .......... .......... .......... .......... 29% 55.5M 0s\n",
            "  5250K .......... .......... .......... .......... .......... 29% 41.7M 0s\n",
            "  5300K .......... .......... .......... .......... .......... 30% 11.8M 0s\n",
            "  5350K .......... .......... .......... .......... .......... 30% 51.0M 0s\n",
            "  5400K .......... .......... .......... .......... .......... 30% 14.5M 0s\n",
            "  5450K .......... .......... .......... .......... .......... 30% 52.3M 0s\n",
            "  5500K .......... .......... .......... .......... .......... 31% 51.5M 0s\n",
            "  5550K .......... .......... .......... .......... .......... 31% 12.6M 0s\n",
            "  5600K .......... .......... .......... .......... .......... 31% 23.6M 0s\n",
            "  5650K .......... .......... .......... .......... .......... 32% 45.8M 0s\n",
            "  5700K .......... .......... .......... .......... .......... 32% 17.1M 0s\n",
            "  5750K .......... .......... .......... .......... .......... 32%  262M 0s\n",
            "  5800K .......... .......... .......... .......... .......... 32%  278M 0s\n",
            "  5850K .......... .......... .......... .......... .......... 33%  279M 0s\n",
            "  5900K .......... .......... .......... .......... .......... 33%  232M 0s\n",
            "  5950K .......... .......... .......... .......... .......... 33%  297M 0s\n",
            "  6000K .......... .......... .......... .......... .......... 34%  281M 0s\n",
            "  6050K .......... .......... .......... .......... .......... 34%  268M 0s\n",
            "  6100K .......... .......... .......... .......... .......... 34%  202M 0s\n",
            "  6150K .......... .......... .......... .......... .......... 34%  250M 0s\n",
            "  6200K .......... .......... .......... .......... .......... 35%  279M 0s\n",
            "  6250K .......... .......... .......... .......... .......... 35%  290M 0s\n",
            "  6300K .......... .......... .......... .......... .......... 35% 12.1M 0s\n",
            "  6350K .......... .......... .......... .......... .......... 35% 15.7M 0s\n",
            "  6400K .......... .......... .......... .......... .......... 36% 10.9M 0s\n",
            "  6450K .......... .......... .......... .......... .......... 36% 54.8M 0s\n",
            "  6500K .......... .......... .......... .......... .......... 36% 44.6M 0s\n",
            "  6550K .......... .......... .......... .......... .......... 37% 9.80M 0s\n",
            "  6600K .......... .......... .......... .......... .......... 37% 46.9M 0s\n",
            "  6650K .......... .......... .......... .......... .......... 37% 58.8M 0s\n",
            "  6700K .......... .......... .......... .......... .......... 37% 67.9M 0s\n",
            "  6750K .......... .......... .......... .......... .......... 38%  284M 0s\n",
            "  6800K .......... .......... .......... .......... .......... 38%  291M 0s\n",
            "  6850K .......... .......... .......... .......... .......... 38%  304M 0s\n",
            "  6900K .......... .......... .......... .......... .......... 39%  267M 0s\n",
            "  6950K .......... .......... .......... .......... .......... 39%  260M 0s\n",
            "  7000K .......... .......... .......... .......... .......... 39% 7.13M 0s\n",
            "  7050K .......... .......... .......... .......... .......... 39% 50.4M 0s\n",
            "  7100K .......... .......... .......... .......... .......... 40% 15.0M 0s\n",
            "  7150K .......... .......... .......... .......... .......... 40% 61.9M 0s\n",
            "  7200K .......... .......... .......... .......... .......... 40% 6.41M 0s\n",
            "  7250K .......... .......... .......... .......... .......... 41% 12.7M 0s\n",
            "  7300K .......... .......... .......... .......... .......... 41%  235M 0s\n",
            "  7350K .......... .......... .......... .......... .......... 41%  279M 0s\n",
            "  7400K .......... .......... .......... .......... .......... 41%  225M 0s\n",
            "  7450K .......... .......... .......... .......... .......... 42% 19.1M 0s\n",
            "  7500K .......... .......... .......... .......... .......... 42% 4.55M 0s\n",
            "  7550K .......... .......... .......... .......... .......... 42% 53.0M 0s\n",
            "  7600K .......... .......... .......... .......... .......... 43% 27.5M 0s\n",
            "  7650K .......... .......... .......... .......... .......... 43% 24.5M 0s\n",
            "  7700K .......... .......... .......... .......... .......... 43% 1.89M 0s\n",
            "  7750K .......... .......... .......... .......... .......... 43%  266M 0s\n",
            "  7800K .......... .......... .......... .......... .......... 44%  253M 0s\n",
            "  7850K .......... .......... .......... .......... .......... 44%  301M 0s\n",
            "  7900K .......... .......... .......... .......... .......... 44%  202M 0s\n",
            "  7950K .......... .......... .......... .......... .......... 44%  269M 0s\n",
            "  8000K .......... .......... .......... .......... .......... 45%  298M 0s\n",
            "  8050K .......... .......... .......... .......... .......... 45%  253M 0s\n",
            "  8100K .......... .......... .......... .......... .......... 45%  243M 0s\n",
            "  8150K .......... .......... .......... .......... .......... 46%  258M 0s\n",
            "  8200K .......... .......... .......... .......... .......... 46%  286M 0s\n",
            "  8250K .......... .......... .......... .......... .......... 46%  274M 0s\n",
            "  8300K .......... .......... .......... .......... .......... 46% 4.24M 0s\n",
            "  8350K .......... .......... .......... .......... .......... 47% 7.73M 0s\n",
            "  8400K .......... .......... .......... .......... .......... 47% 50.1M 0s\n",
            "  8450K .......... .......... .......... .......... .......... 47% 17.1M 0s\n",
            "  8500K .......... .......... .......... .......... .......... 48% 8.38M 0s\n",
            "  8550K .......... .......... .......... .......... .......... 48% 5.95M 0s\n",
            "  8600K .......... .......... .......... .......... .......... 48% 55.5M 0s\n",
            "  8650K .......... .......... .......... .......... .......... 48% 51.3M 0s\n",
            "  8700K .......... .......... .......... .......... .......... 49% 5.84M 0s\n",
            "  8750K .......... .......... .......... .......... .......... 49% 73.7M 0s\n",
            "  8800K .......... .......... .......... .......... .......... 49%  281M 0s\n",
            "  8850K .......... .......... .......... .......... .......... 50%  328M 0s\n",
            "  8900K .......... .......... .......... .......... .......... 50%  210M 0s\n",
            "  8950K .......... .......... .......... .......... .......... 50%  253M 0s\n",
            "  9000K .......... .......... .......... .......... .......... 50%  278M 0s\n",
            "  9050K .......... .......... .......... .......... .......... 51%  271M 0s\n",
            "  9100K .......... .......... .......... .......... .......... 51%  240M 0s\n",
            "  9150K .......... .......... .......... .......... .......... 51%  335M 0s\n",
            "  9200K .......... .......... .......... .......... .......... 52% 5.98M 0s\n",
            "  9250K .......... .......... .......... .......... .......... 52% 17.7M 0s\n",
            "  9300K .......... .......... .......... .......... .......... 52% 49.4M 0s\n",
            "  9350K .......... .......... .......... .......... .......... 52% 5.63M 0s\n",
            "  9400K .......... .......... .......... .......... .......... 53% 55.7M 0s\n",
            "  9450K .......... .......... .......... .......... .......... 53% 6.07M 0s\n",
            "  9500K .......... .......... .......... .......... .......... 53% 39.2M 0s\n",
            "  9550K .......... .......... .......... .......... .......... 53% 55.1M 0s\n",
            "  9600K .......... .......... .......... .......... .......... 54% 56.8M 0s\n",
            "  9650K .......... .......... .......... .......... .......... 54%  249M 0s\n",
            "  9700K .......... .......... .......... .......... .......... 54%  203M 0s\n",
            "  9750K .......... .......... .......... .......... .......... 55%  288M 0s\n",
            "  9800K .......... .......... .......... .......... .......... 55%  241M 0s\n",
            "  9850K .......... .......... .......... .......... .......... 55%  289M 0s\n",
            "  9900K .......... .......... .......... .......... .......... 55%  227M 0s\n",
            "  9950K .......... .......... .......... .......... .......... 56%  281M 0s\n",
            " 10000K .......... .......... .......... .......... .......... 56%  310M 0s\n",
            " 10050K .......... .......... .......... .......... .......... 56% 4.28M 0s\n",
            " 10100K .......... .......... .......... .......... .......... 57% 12.7M 0s\n",
            " 10150K .......... .......... .......... .......... .......... 57% 52.0M 0s\n",
            " 10200K .......... .......... .......... .......... .......... 57% 51.8M 0s\n",
            " 10250K .......... .......... .......... .......... .......... 57% 56.6M 0s\n",
            " 10300K .......... .......... .......... .......... .......... 58% 6.03M 0s\n",
            " 10350K .......... .......... .......... .......... .......... 58% 52.2M 0s\n",
            " 10400K .......... .......... .......... .......... .......... 58% 63.1M 0s\n",
            " 10450K .......... .......... .......... .......... .......... 59%  236M 0s\n",
            " 10500K .......... .......... .......... .......... .......... 59%  243M 0s\n",
            " 10550K .......... .......... .......... .......... .......... 59%  312M 0s\n",
            " 10600K .......... .......... .......... .......... .......... 59%  299M 0s\n",
            " 10650K .......... .......... .......... .......... .......... 60%  281M 0s\n",
            " 10700K .......... .......... .......... .......... .......... 60%  244M 0s\n",
            " 10750K .......... .......... .......... .......... .......... 60%  293M 0s\n",
            " 10800K .......... .......... .......... .......... .......... 61%  251M 0s\n",
            " 10850K .......... .......... .......... .......... .......... 61%  281M 0s\n",
            " 10900K .......... .......... .......... .......... .......... 61%  228M 0s\n",
            " 10950K .......... .......... .......... .......... .......... 61%  246M 0s\n",
            " 11000K .......... .......... .......... .......... .......... 62%  267M 0s\n",
            " 11050K .......... .......... .......... .......... .......... 62% 4.95M 0s\n",
            " 11100K .......... .......... .......... .......... .......... 62% 12.2M 0s\n",
            " 11150K .......... .......... .......... .......... .......... 62% 16.7M 0s\n",
            " 11200K .......... .......... .......... .......... .......... 63% 24.4M 0s\n",
            " 11250K .......... .......... .......... .......... .......... 63% 55.0M 0s\n",
            " 11300K .......... .......... .......... .......... .......... 63% 13.6M 0s\n",
            " 11350K .......... .......... .......... .......... .......... 64% 24.7M 0s\n",
            " 11400K .......... .......... .......... .......... .......... 64% 55.0M 0s\n",
            " 11450K .......... .......... .......... .......... .......... 64% 12.2M 0s\n",
            " 11500K .......... .......... .......... .......... .......... 64% 47.6M 0s\n",
            " 11550K .......... .......... .......... .......... .......... 65% 71.9M 0s\n",
            " 11600K .......... .......... .......... .......... .......... 65%  319M 0s\n",
            " 11650K .......... .......... .......... .......... .......... 65% 19.7M 0s\n",
            " 11700K .......... .......... .......... .......... .......... 66% 55.8M 0s\n",
            " 11750K .......... .......... .......... .......... .......... 66%  298M 0s\n",
            " 11800K .......... .......... .......... .......... .......... 66%  267M 0s\n",
            " 11850K .......... .......... .......... .......... .......... 66%  146M 0s\n",
            " 11900K .......... .......... .......... .......... .......... 67% 11.5M 0s\n",
            " 11950K .......... .......... .......... .......... .......... 67% 58.1M 0s\n",
            " 12000K .......... .......... .......... .......... .......... 67% 59.5M 0s\n",
            " 12050K .......... .......... .......... .......... .......... 68% 48.4M 0s\n",
            " 12100K .......... .......... .......... .......... .......... 68%  143M 0s\n",
            " 12150K .......... .......... .......... .......... .......... 68% 18.8M 0s\n",
            " 12200K .......... .......... .......... .......... .......... 68%  282M 0s\n",
            " 12250K .......... .......... .......... .......... .......... 69% 18.3M 0s\n",
            " 12300K .......... .......... .......... .......... .......... 69% 91.1M 0s\n",
            " 12350K .......... .......... .......... .......... .......... 69%  277M 0s\n",
            " 12400K .......... .......... .......... .......... .......... 69%  272M 0s\n",
            " 12450K .......... .......... .......... .......... .......... 70%  280M 0s\n",
            " 12500K .......... .......... .......... .......... .......... 70%  215M 0s\n",
            " 12550K .......... .......... .......... .......... .......... 70%  280M 0s\n",
            " 12600K .......... .......... .......... .......... .......... 71%  287M 0s\n",
            " 12650K .......... .......... .......... .......... .......... 71% 14.0M 0s\n",
            " 12700K .......... .......... .......... .......... .......... 71% 50.9M 0s\n",
            " 12750K .......... .......... .......... .......... .......... 71% 12.3M 0s\n",
            " 12800K .......... .......... .......... .......... .......... 72% 58.1M 0s\n",
            " 12850K .......... .......... .......... .......... .......... 72% 57.8M 0s\n",
            " 12900K .......... .......... .......... .......... .......... 72% 14.6M 0s\n",
            " 12950K .......... .......... .......... .......... .......... 73%  279M 0s\n",
            " 13000K .......... .......... .......... .......... .......... 73%  337M 0s\n",
            " 13050K .......... .......... .......... .......... .......... 73%  264M 0s\n",
            " 13100K .......... .......... .......... .......... .......... 73%  266M 0s\n",
            " 13150K .......... .......... .......... .......... .......... 74%  293M 0s\n",
            " 13200K .......... .......... .......... .......... .......... 74%  307M 0s\n",
            " 13250K .......... .......... .......... .......... .......... 74%  314M 0s\n",
            " 13300K .......... .......... .......... .......... .......... 75%  238M 0s\n",
            " 13350K .......... .......... .......... .......... .......... 75%  247M 0s\n",
            " 13400K .......... .......... .......... .......... .......... 75%  300M 0s\n",
            " 13450K .......... .......... .......... .......... .......... 75%  262M 0s\n",
            " 13500K .......... .......... .......... .......... .......... 76%  247M 0s\n",
            " 13550K .......... .......... .......... .......... .......... 76%  292M 0s\n",
            " 13600K .......... .......... .......... .......... .......... 76% 5.89M 0s\n",
            " 13650K .......... .......... .......... .......... .......... 77% 56.4M 0s\n",
            " 13700K .......... .......... .......... .......... .......... 77% 49.6M 0s\n",
            " 13750K .......... .......... .......... .......... .......... 77% 5.80M 0s\n",
            " 13800K .......... .......... .......... .......... .......... 77% 53.8M 0s\n",
            " 13850K .......... .......... .......... .......... .......... 78% 51.9M 0s\n",
            " 13900K .......... .......... .......... .......... .......... 78% 8.16M 0s\n",
            " 13950K .......... .......... .......... .......... .......... 78% 55.7M 0s\n",
            " 14000K .......... .......... .......... .......... .......... 78% 55.6M 0s\n",
            " 14050K .......... .......... .......... .......... .......... 79% 9.11M 0s\n",
            " 14100K .......... .......... .......... .......... .......... 79% 79.8M 0s\n",
            " 14150K .......... .......... .......... .......... .......... 79%  280M 0s\n",
            " 14200K .......... .......... .......... .......... .......... 80%  253M 0s\n",
            " 14250K .......... .......... .......... .......... .......... 80%  288M 0s\n",
            " 14300K .......... .......... .......... .......... .......... 80%  246M 0s\n",
            " 14350K .......... .......... .......... .......... .......... 80%  277M 0s\n",
            " 14400K .......... .......... .......... .......... .......... 81%  266M 0s\n",
            " 14450K .......... .......... .......... .......... .......... 81%  295M 0s\n",
            " 14500K .......... .......... .......... .......... .......... 81%  249M 0s\n",
            " 14550K .......... .......... .......... .......... .......... 82%  325M 0s\n",
            " 14600K .......... .......... .......... .......... .......... 82%  305M 0s\n",
            " 14650K .......... .......... .......... .......... .......... 82%  142M 0s\n",
            " 14700K .......... .......... .......... .......... .......... 82% 7.80M 0s\n",
            " 14750K .......... .......... .......... .......... .......... 83% 54.7M 0s\n",
            " 14800K .......... .......... .......... .......... .......... 83% 56.6M 0s\n",
            " 14850K .......... .......... .......... .......... .......... 83% 6.65M 0s\n",
            " 14900K .......... .......... .......... .......... .......... 84% 56.6M 0s\n",
            " 14950K .......... .......... .......... .......... .......... 84% 56.0M 0s\n",
            " 15000K .......... .......... .......... .......... .......... 84% 54.4M 0s\n",
            " 15050K .......... .......... .......... .......... .......... 84% 55.1M 0s\n",
            " 15100K .......... .......... .......... .......... .......... 85% 5.92M 0s\n",
            " 15150K .......... .......... .......... .......... .......... 85% 72.9M 0s\n",
            " 15200K .......... .......... .......... .......... .......... 85%  246M 0s\n",
            " 15250K .......... .......... .......... .......... .......... 86%  286M 0s\n",
            " 15300K .......... .......... .......... .......... .......... 86%  246M 0s\n",
            " 15350K .......... .......... .......... .......... .......... 86%  292M 0s\n",
            " 15400K .......... .......... .......... .......... .......... 86%  312M 0s\n",
            " 15450K .......... .......... .......... .......... .......... 87%  261M 0s\n",
            " 15500K .......... .......... .......... .......... .......... 87%  244M 0s\n",
            " 15550K .......... .......... .......... .......... .......... 87%  297M 0s\n",
            " 15600K .......... .......... .......... .......... .......... 87%  288M 0s\n",
            " 15650K .......... .......... .......... .......... .......... 88%  294M 0s\n",
            " 15700K .......... .......... .......... .......... .......... 88%  260M 0s\n",
            " 15750K .......... .......... .......... .......... .......... 88%  243M 0s\n",
            " 15800K .......... .......... .......... .......... .......... 89%  297M 0s\n",
            " 15850K .......... .......... .......... .......... .......... 89%  282M 0s\n",
            " 15900K .......... .......... .......... .......... .......... 89%  264M 0s\n",
            " 15950K .......... .......... .......... .......... .......... 89%  313M 0s\n",
            " 16000K .......... .......... .......... .......... .......... 90%  149M 0s\n",
            " 16050K .......... .......... .......... .......... .......... 90% 4.28M 0s\n",
            " 16100K .......... .......... .......... .......... .......... 90% 48.9M 0s\n",
            " 16150K .......... .......... .......... .......... .......... 91% 6.11M 0s\n",
            " 16200K .......... .......... .......... .......... .......... 91% 57.6M 0s\n",
            " 16250K .......... .......... .......... .......... .......... 91%  257M 0s\n",
            " 16300K .......... .......... .......... .......... .......... 91%  242M 0s\n",
            " 16350K .......... .......... .......... .......... .......... 92%  271M 0s\n",
            " 16400K .......... .......... .......... .......... .......... 92%  257M 0s\n",
            " 16450K .......... .......... .......... .......... .......... 92%  289M 0s\n",
            " 16500K .......... .......... .......... .......... .......... 93%  227M 0s\n",
            " 16550K .......... .......... .......... .......... .......... 93%  293M 0s\n",
            " 16600K .......... .......... .......... .......... .......... 93%  286M 0s\n",
            " 16650K .......... .......... .......... .......... .......... 93% 90.5M 0s\n",
            " 16700K .......... .......... .......... .......... .......... 94% 46.9M 0s\n",
            " 16750K .......... .......... .......... .......... .......... 94% 6.04M 0s\n",
            " 16800K .......... .......... .......... .......... .......... 94% 58.3M 0s\n",
            " 16850K .......... .......... .......... .......... .......... 95% 56.7M 0s\n",
            " 16900K .......... .......... .......... .......... .......... 95% 51.0M 0s\n",
            " 16950K .......... .......... .......... .......... .......... 95% 54.1M 0s\n",
            " 17000K .......... .......... .......... .......... .......... 95% 81.7M 0s\n",
            " 17050K .......... .......... .......... .......... .......... 96%  299M 0s\n",
            " 17100K .......... .......... .......... .......... .......... 96% 5.87M 0s\n",
            " 17150K .......... .......... .......... .......... .......... 96% 67.5M 0s\n",
            " 17200K .......... .......... .......... .......... .......... 96%  264M 0s\n",
            " 17250K .......... .......... .......... .......... .......... 97%  271M 0s\n",
            " 17300K .......... .......... .......... .......... .......... 97%  244M 0s\n",
            " 17350K .......... .......... .......... .......... .......... 97%  298M 0s\n",
            " 17400K .......... .......... .......... .......... .......... 98%  289M 0s\n",
            " 17450K .......... .......... .......... .......... .......... 98%  256M 0s\n",
            " 17500K .......... .......... .......... .......... .......... 98%  243M 0s\n",
            " 17550K .......... .......... .......... .......... .......... 98%  298M 0s\n",
            " 17600K .......... .......... .......... .......... .......... 99%  272M 0s\n",
            " 17650K .......... .......... .......... .......... .......... 99%  271M 0s\n",
            " 17700K .......... .......... .......... .......... .......... 99%  246M 0s\n",
            " 17750K .......... .......... .......... ......               100% 72.5M=0.6s\n",
            "\n",
            "2024-06-19 19:22:28 (28.9 MB/s) - ‘hmmer-3.3.2.tar.gz’ saved [18213049/18213049]\n",
            "\n"
          ]
        }
      ],
      "source": [
        "%%bash\n",
        "wget http://eddylab.org/software/hmmer/hmmer-3.3.2.tar.gz\n",
        "tar xf hmmer-3.3.2.tar.gz\n",
        "cd hmmer-3.3.2"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "%%bash\n",
        "cd hmmer-3.3.2\n",
        "\n",
        "./configure\n",
        "make\n",
        "make install"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "wodGz6biJPCb",
        "outputId": "15126950-9311-41c5-80dd-1807b2c5ac7a"
      },
      "execution_count": 3,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "configure: Configuring HMMER3 for your system.\n",
            "checking build system type... x86_64-pc-linux-gnu\n",
            "checking host system type... x86_64-pc-linux-gnu\n",
            "checking whether to compile using MPI... no\n",
            "checking for gcc... gcc\n",
            "checking whether the C compiler works... yes\n",
            "checking for C compiler default output file name... a.out\n",
            "checking for suffix of executables... \n",
            "checking whether we are cross compiling... no\n",
            "checking for suffix of object files... o\n",
            "checking whether we are using the GNU C compiler... yes\n",
            "checking whether gcc accepts -g... yes\n",
            "checking for gcc option to accept ISO C89... none needed\n",
            "checking for gcc option to accept ISO C99... none needed\n",
            "checking for gcc option to accept ISO Standard C... (cached) none needed\n",
            "checking how to run the C preprocessor... gcc -E\n",
            "checking for a BSD-compatible install... /usr/bin/install -c\n",
            "checking for strip... strip\n",
            "checking for ranlib... ranlib\n",
            "checking for ar... /usr/bin/ar\n",
            "checking whether ln -s works... yes\n",
            "checking for a sed that does not truncate output... /usr/bin/sed\n",
            "checking for grep that handles long lines and -e... /usr/bin/grep\n",
            "checking for egrep... /usr/bin/grep -E\n",
            "checking whether gcc is Clang... no\n",
            "checking whether pthreads work with -pthread... yes\n",
            "checking for joinable pthread attribute... PTHREAD_CREATE_JOINABLE\n",
            "checking whether more special flags are required for pthreads... no\n",
            "checking for PTHREAD_PRIO_INHERIT... yes\n",
            "checking whether gcc can compile our Altivec/VMX code... no\n",
            "checking whether gcc can compile our SSE code... yes\n",
            "configure: Activating Intel/AMD SSE vector DP implementation\n",
            "checking whether _MM_SET_FLUSH_ZERO_MODE is supported... yes\n",
            "checking compiler support --start-group... yes\n",
            "checking for gzip... /usr/bin/gzip\n",
            "checking for python3... yes\n",
            "checking for library containing clock_gettime... none required\n",
            "checking for ANSI C header files... yes\n",
            "checking for sys/types.h... yes\n",
            "checking for sys/stat.h... yes\n",
            "checking for stdlib.h... yes\n",
            "checking for string.h... yes\n",
            "checking for memory.h... yes\n",
            "checking for strings.h... yes\n",
            "checking for inttypes.h... yes\n",
            "checking for stdint.h... yes\n",
            "checking for unistd.h... yes\n",
            "checking endian.h usability... yes\n",
            "checking endian.h presence... yes\n",
            "checking for endian.h... yes\n",
            "checking for inttypes.h... (cached) yes\n",
            "checking for stdint.h... (cached) yes\n",
            "checking for unistd.h... (cached) yes\n",
            "checking for sys/types.h... (cached) yes\n",
            "checking netinet/in.h usability... yes\n",
            "checking netinet/in.h presence... yes\n",
            "checking for netinet/in.h... yes\n",
            "checking sys/param.h usability... yes\n",
            "checking sys/param.h presence... yes\n",
            "checking for sys/param.h... yes\n",
            "checking for sys/sysctl.h... no\n",
            "checking for uint8_t... yes\n",
            "checking for uint16_t... yes\n",
            "checking for uint32_t... yes\n",
            "checking for uint64_t... yes\n",
            "checking for off_t... yes\n",
            "checking whether byte ordering is bigendian... no\n",
            "checking for __attribute__((noreturn))... yes\n",
            "checking for __attribute__((format))... yes\n",
            "checking for mkstemp... yes\n",
            "checking for popen... yes\n",
            "checking for putenv... yes\n",
            "checking for strcasecmp... yes\n",
            "checking for strsep... yes\n",
            "checking for times... yes\n",
            "checking for getpid... yes\n",
            "checking for sysctl... no\n",
            "checking for sysconf... yes\n",
            "checking for getcwd... yes\n",
            "checking for chmod... yes\n",
            "checking for stat... yes\n",
            "checking for fstat... yes\n",
            "checking for erfc... no\n",
            "checking for library containing ntohs... none required\n",
            "checking for library containing ntohl... none required\n",
            "checking for library containing htons... none required\n",
            "checking for library containing htonl... none required\n",
            "checking for library containing socket... none required\n",
            "checking for library containing inet_pton... none required\n",
            "checking for _LARGEFILE_SOURCE value needed for large files... no\n",
            "checking for special C compiler options needed for large files... no\n",
            "checking for _FILE_OFFSET_BITS value needed for large files... no\n",
            "configure: creating ./config.status\n",
            "config.status: creating Makefile\n",
            "config.status: creating src/Makefile\n",
            "config.status: creating testsuite/Makefile\n",
            "config.status: creating profmark/Makefile\n",
            "config.status: creating src/impl_sse/Makefile\n",
            "config.status: creating documentation/Makefile\n",
            "config.status: creating documentation/man/Makefile\n",
            "config.status: creating documentation/userguide/Makefile\n",
            "config.status: creating documentation/userguide/inclusions/Makefile\n",
            "config.status: creating libdivsufsort/Makefile\n",
            "config.status: creating easel/Makefile\n",
            "config.status: creating easel/miniapps/Makefile\n",
            "config.status: creating easel/testsuite/Makefile\n",
            "config.status: creating easel/documentation/Makefile\n",
            "config.status: creating documentation/userguide/titlepage.tex\n",
            "config.status: creating documentation/userguide/titlepage_daemon.tex\n",
            "config.status: creating documentation/userguide/copyright.tex\n",
            "config.status: creating documentation/man/alimask.man\n",
            "config.status: creating documentation/man/hmmalign.man\n",
            "config.status: creating documentation/man/hmmbuild.man\n",
            "config.status: creating documentation/man/hmmc2.man\n",
            "config.status: creating documentation/man/hmmconvert.man\n",
            "config.status: creating documentation/man/hmmemit.man\n",
            "config.status: creating documentation/man/hmmer.man\n",
            "config.status: creating documentation/man/hmmfetch.man\n",
            "config.status: creating documentation/man/hmmlogo.man\n",
            "config.status: creating documentation/man/hmmpgmd.man\n",
            "config.status: creating documentation/man/hmmpgmd_shard.man\n",
            "config.status: creating documentation/man/hmmpress.man\n",
            "config.status: creating documentation/man/hmmscan.man\n",
            "config.status: creating documentation/man/hmmsearch.man\n",
            "config.status: creating documentation/man/hmmsim.man\n",
            "config.status: creating documentation/man/hmmstat.man\n",
            "config.status: creating documentation/man/jackhmmer.man\n",
            "config.status: creating documentation/man/makehmmerdb.man\n",
            "config.status: creating documentation/man/nhmmer.man\n",
            "config.status: creating documentation/man/nhmmscan.man\n",
            "config.status: creating documentation/man/phmmer.man\n",
            "config.status: creating easel/miniapps/esl-afetch.man\n",
            "config.status: creating easel/miniapps/esl-alimanip.man\n",
            "config.status: creating easel/miniapps/esl-alimap.man\n",
            "config.status: creating easel/miniapps/esl-alimask.man\n",
            "config.status: creating easel/miniapps/esl-alimerge.man\n",
            "config.status: creating easel/miniapps/esl-alipid.man\n",
            "config.status: creating easel/miniapps/esl-alirev.man\n",
            "config.status: creating easel/miniapps/esl-alistat.man\n",
            "config.status: creating easel/miniapps/esl-compalign.man\n",
            "config.status: creating easel/miniapps/esl-compstruct.man\n",
            "config.status: creating easel/miniapps/esl-construct.man\n",
            "config.status: creating easel/miniapps/esl-histplot.man\n",
            "config.status: creating easel/miniapps/esl-mask.man\n",
            "config.status: creating easel/miniapps/esl-mixdchlet.man\n",
            "config.status: creating easel/miniapps/esl-reformat.man\n",
            "config.status: creating easel/miniapps/esl-selectn.man\n",
            "config.status: creating easel/miniapps/esl-seqrange.man\n",
            "config.status: creating easel/miniapps/esl-seqstat.man\n",
            "config.status: creating easel/miniapps/esl-sfetch.man\n",
            "config.status: creating easel/miniapps/esl-shuffle.man\n",
            "config.status: creating easel/miniapps/esl-ssdraw.man\n",
            "config.status: creating easel/miniapps/esl-translate.man\n",
            "config.status: creating easel/miniapps/esl-weight.man\n",
            "config.status: creating easel/decoy_config.h\n",
            "config.status: creating src/p7_config.h\n",
            "config.status: creating easel/esl_config.h\n",
            "config.status: creating libdivsufsort/divsufsort.h\n",
            "config.status: linking src/impl_sse to src/impl\n",
            "\n",
            "\n",
            "HMMER configuration:\n",
            "   compiler:             gcc -O3   -pthread \n",
            "   host:                 x86_64-pc-linux-gnu\n",
            "   linker:               \n",
            "   libraries:              \n",
            "   DP implementation:    sse\n",
            "\n",
            "Now do 'make'  to build HMMER, and optionally:\n",
            "       'make check'  to run self tests,\n",
            "       'make install'  to install programs and man pages,\n",
            "       '(cd easel; make install)'  to install Easel tools.\n",
            "\n",
            "     SUBDIR easel\n",
            "     CC easel.o\n",
            "     CC esl_alloc.o\n",
            "     CC esl_alphabet.o\n",
            "     CC esl_arr2.o\n",
            "     CC esl_arr3.o\n",
            "     CC esl_bitfield.o\n",
            "     CC esl_buffer.o\n",
            "     CC esl_cluster.o\n",
            "     CC esl_composition.o\n",
            "     CC esl_cpu.o\n",
            "     CC esl_dirichlet.o\n",
            "     CC esl_distance.o\n",
            "     CC esl_dmatrix.o\n",
            "     CC esl_dsqdata.o\n",
            "     CC esl_exponential.o\n",
            "     CC esl_fileparser.o\n",
            "     CC esl_gamma.o\n",
            "     CC esl_gencode.o\n",
            "     CC esl_getopts.o\n",
            "     CC esl_gev.o\n",
            "     CC esl_graph.o\n",
            "     CC esl_gumbel.o\n",
            "     CC esl_heap.o\n",
            "     CC esl_histogram.o\n",
            "     CC esl_hmm.o\n",
            "     CC esl_huffman.o\n",
            "     CC esl_hyperexp.o\n",
            "     CC esl_json.o\n",
            "     CC esl_keyhash.o\n",
            "     CC esl_matrixops.o\n",
            "     CC esl_mem.o\n",
            "     CC esl_minimizer.o\n",
            "     CC esl_mixdchlet.o\n",
            "     CC esl_mixgev.o\n",
            "     CC esl_mpi.o\n",
            "     CC esl_msa.o\n",
            "     CC esl_msacluster.o\n",
            "     CC esl_msafile.o\n",
            "     CC esl_msafile2.o\n",
            "     CC esl_msafile_a2m.o\n",
            "     CC esl_msafile_afa.o\n",
            "     CC esl_msafile_clustal.o\n",
            "     CC esl_msafile_phylip.o\n",
            "     CC esl_msafile_psiblast.o\n",
            "     CC esl_msafile_selex.o\n",
            "     CC esl_msafile_stockholm.o\n",
            "     CC esl_msashuffle.o\n",
            "     CC esl_msaweight.o\n",
            "     CC esl_normal.o\n",
            "     CC esl_paml.o\n",
            "     CC esl_quicksort.o\n",
            "     CC esl_random.o\n",
            "     CC esl_rand64.o\n",
            "     CC esl_randomseq.o\n",
            "     CC esl_ratematrix.o\n",
            "     CC esl_recorder.o\n",
            "     CC esl_red_black.o\n",
            "     CC esl_regexp.o\n",
            "     CC esl_rootfinder.o\n",
            "     CC esl_scorematrix.o\n",
            "     CC esl_sq.o\n",
            "     CC esl_sqio.o\n",
            "     CC esl_sqio_ascii.o\n",
            "     CC esl_sqio_ncbi.o\n",
            "     CC esl_ssi.o\n",
            "     CC esl_stack.o\n",
            "     CC esl_stats.o\n",
            "     CC esl_stopwatch.o\n",
            "     CC esl_stretchexp.o\n",
            "     CC esl_subcmd.o\n",
            "     CC esl_threads.o\n",
            "     CC esl_tree.o\n",
            "     CC esl_varint.o\n",
            "     CC esl_vectorops.o\n",
            "     CC esl_weibull.o\n",
            "     CC esl_workqueue.o\n",
            "     CC esl_wuss.o\n",
            "     CC esl_sse.o\n",
            "     CC esl_avx.o\n",
            "     CC esl_avx512.o\n",
            "     CC esl_neon.o\n",
            "     CC esl_vmx.o\n",
            "     AR libeasel.a\n",
            "     SUBDIR miniapps\n",
            "     GEN esl-afetch\n",
            "     GEN esl-alimanip\n",
            "     GEN esl-alimap\n",
            "     GEN esl-alimask\n",
            "     GEN esl-alimerge\n",
            "     GEN esl-alipid\n",
            "     GEN esl-alirev\n",
            "     GEN esl-alistat\n",
            "     GEN esl-compalign\n",
            "     GEN esl-compstruct\n",
            "     GEN esl-construct\n",
            "     GEN esl-histplot\n",
            "     GEN esl-mask\n",
            "     GEN esl-mixdchlet\n",
            "     GEN esl-reformat\n",
            "     GEN esl-selectn\n",
            "     GEN esl-seqrange\n",
            "     GEN esl-seqstat\n",
            "     GEN esl-sfetch\n",
            "     GEN esl-shuffle\n",
            "     GEN esl-ssdraw\n",
            "     GEN esl-translate\n",
            "     GEN esl-weight\n",
            "     CC cmd_alistat.o\n",
            "     CC cmd_downsample.o\n",
            "     CC cmd_filter.o\n",
            "     CC cmd_index.o\n",
            "     GEN easel\n",
            "     SUBDIR libdivsufsort\n",
            "     CC divsufsort.o\n",
            "     AR libdivsufsort.a\n",
            "     SUBDIR src\n",
            "     CC alimask.o\n",
            "     CC build.o\n",
            "     CC cachedb.o\n",
            "     CC cachedb_shard.o\n",
            "     CC emit.o\n",
            "     CC errors.o\n",
            "     CC evalues.o\n",
            "     CC eweight.o\n",
            "     CC generic_decoding.o\n",
            "     CC generic_fwdback.o\n",
            "     CC generic_fwdback_chk.o\n",
            "     CC generic_fwdback_banded.o\n",
            "     CC generic_null2.o\n",
            "     CC generic_msv.o\n",
            "     CC generic_optacc.o\n",
            "     CC generic_stotrace.o\n",
            "     CC generic_viterbi.o\n",
            "     CC generic_vtrace.o\n",
            "     CC h2_io.o\n",
            "     CC heatmap.o\n",
            "     CC hmmlogo.o\n",
            "     CC hmmdmstr.o\n",
            "     CC hmmdmstr_shard.o\n",
            "     CC hmmd_search_status.o\n",
            "     CC hmmdwrkr.o\n",
            "     CC hmmdwrkr_shard.o\n",
            "     CC hmmdutils.o\n",
            "     CC hmmer.o\n",
            "     CC logsum.o\n",
            "     CC modelconfig.o\n",
            "     CC modelstats.o\n",
            "     CC mpisupport.o\n",
            "     CC seqmodel.o\n",
            "     CC tracealign.o\n",
            "     CC p7_alidisplay.o\n",
            "     CC p7_bg.o\n",
            "     CC p7_builder.o\n",
            "     CC p7_domain.o\n",
            "     CC p7_domaindef.o\n",
            "     CC p7_gbands.o\n",
            "     CC p7_gmx.o\n",
            "     CC p7_gmxb.o\n",
            "     CC p7_gmxchk.o\n",
            "     CC p7_hit.o\n",
            "     CC p7_hmm.o\n",
            "     CC p7_hmmcache.o\n",
            "     CC p7_hmmd_search_stats.o\n",
            "     CC p7_hmmfile.o\n",
            "     CC p7_hmmwindow.o\n",
            "     CC p7_pipeline.o\n",
            "     CC p7_prior.o\n",
            "     CC p7_profile.o\n",
            "     CC p7_spensemble.o\n",
            "     CC p7_tophits.o\n",
            "     CC p7_trace.o\n",
            "     CC p7_scoredata.o\n",
            "     CC hmmpgmd2msa.o\n",
            "     CC fm_alphabet.o\n",
            "     CC fm_general.o\n",
            "     CC fm_sse.o\n",
            "     CC fm_ssv.o\n",
            "     AR libhmmer-src.stamp\n",
            "     SUBDIR impl_sse\n",
            "     CC decoding.o\n",
            "     CC fwdback.o\n",
            "     CC io.o\n",
            "     CC ssvfilter.o\n",
            "     CC msvfilter.o\n",
            "     CC null2.o\n",
            "     CC optacc.o\n",
            "     CC stotrace.o\n",
            "     CC vitfilter.o\n",
            "     CC p7_omx.o\n",
            "     CC p7_oprofile.o\n",
            "     CC mpi.o\n",
            "     AR libhmmer-impl.stamp\n",
            "     GEN alimask\n",
            "     CC hmmalign.o\n",
            "     GEN hmmalign\n",
            "     CC hmmbuild.o\n",
            "     GEN hmmbuild\n",
            "     CC hmmconvert.o\n",
            "     GEN hmmconvert\n",
            "     CC hmmemit.o\n",
            "     GEN hmmemit\n",
            "     CC hmmfetch.o\n",
            "     GEN hmmfetch\n",
            "     GEN hmmlogo\n",
            "     CC hmmpgmd.o\n",
            "     GEN hmmpgmd\n",
            "     CC hmmpgmd_shard.o\n",
            "     GEN hmmpgmd_shard\n",
            "     CC hmmpress.o\n",
            "     GEN hmmpress\n",
            "     CC hmmscan.o\n",
            "     GEN hmmscan\n",
            "     CC hmmsearch.o\n",
            "     GEN hmmsearch\n",
            "     CC hmmsim.o\n",
            "     GEN hmmsim\n",
            "     CC hmmstat.o\n",
            "     GEN hmmstat\n",
            "     CC jackhmmer.o\n",
            "     GEN jackhmmer\n",
            "     CC phmmer.o\n",
            "     GEN phmmer\n",
            "     CC nhmmer.o\n",
            "     GEN nhmmer\n",
            "     CC nhmmscan.o\n",
            "     GEN nhmmscan\n",
            "     CC makehmmerdb.o\n",
            "     GEN makehmmerdb\n",
            "     CC hmmc2.o\n",
            "     GEN hmmc2\n",
            "     CC hmmerfm-exactmatch.o\n",
            "     GEN hmmerfm-exactmatch\n",
            "     SUBDIR profmark\n",
            "     CC create-profmark.o\n",
            "     GEN create-profmark\n",
            "     CC rocplot.o\n",
            "     GEN rocplot\n",
            "     SUBDIR easel\n",
            "     SUBDIR miniapps\n",
            "     SUBDIR libdivsufsort\n",
            "     SUBDIR src\n",
            "     SUBDIR impl_sse\n",
            "     SUBDIR profmark\n",
            "/usr/bin/install -c -d /usr/local/bin\n",
            "/usr/bin/install -c -d /usr/local/share/man/man1\n",
            "make -C src install\n",
            "make[1]: Entering directory '/content/hmmer-3.3.2/src'\n",
            "for file in alimask hmmalign hmmbuild hmmconvert hmmemit hmmfetch hmmlogo hmmpgmd hmmpgmd_shard hmmpress hmmscan hmmsearch hmmsim hmmstat jackhmmer phmmer nhmmer nhmmscan makehmmerdb; do \\\n",
            "   /usr/bin/install -c -m 0755 $file /usr/local/bin/ ;\\\n",
            "done\n",
            "make[1]: Leaving directory '/content/hmmer-3.3.2/src'\n",
            "make -C documentation install\n",
            "make[1]: Entering directory '/content/hmmer-3.3.2/documentation'\n",
            "make -C man install\n",
            "make[2]: Entering directory '/content/hmmer-3.3.2/documentation/man'\n",
            "for file in hmmer hmmalign hmmbuild hmmc2 hmmconvert hmmemit hmmfetch hmmlogo hmmpgmd hmmpgmd_shard hmmpress hmmscan hmmsearch hmmsim hmmstat jackhmmer makehmmerdb phmmer nhmmer nhmmscan alimask; do \\\n",
            "   /usr/bin/install -c -m 0664 $file.man /usr/local/share/man/man1/${file}.1 ;\\\n",
            "done\n",
            "make[2]: Leaving directory '/content/hmmer-3.3.2/documentation/man'\n",
            "make[1]: Leaving directory '/content/hmmer-3.3.2/documentation'\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "In file included from hmmbuild.c:13:\n",
            "In function ‘usual_master’,\n",
            "    inlined from ‘main’ at hmmbuild.c:463:7:\n",
            "../easel/easel.h:164:17: warning: argument 1 range [18446744022169944064, 18446744073709551592] exceeds maximum object size 9223372036854775807 [-Walloc-size-larger-than=]\n",
            "  164 |     if ( ((p) = malloc(size)) == NULL)  { \\\n",
            "      |                 ^~~~~~~~~~~~\n",
            "hmmbuild.c:561:3: note: in expansion of macro ‘ESL_ALLOC’\n",
            "  561 |   ESL_ALLOC(info, sizeof(*info) * infocnt);\n",
            "      |   ^~~~~~~~~\n",
            "In file included from hmmbuild.c:6:\n",
            "hmmbuild.c: In function ‘main’:\n",
            "/usr/include/stdlib.h:540:14: note: in a call to allocation function ‘malloc’ declared here\n",
            "  540 | extern void *malloc (size_t __size) __THROW __attribute_malloc__\n",
            "      |              ^~~~~~\n",
            "In file included from hmmscan.c:14:\n",
            "In function ‘serial_master’,\n",
            "    inlined from ‘main’ at hmmscan.c:302:16:\n",
            "../easel/easel.h:164:17: warning: argument 1 range [18446743987810205696, 18446744073709551576] exceeds maximum object size 9223372036854775807 [-Walloc-size-larger-than=]\n",
            "  164 |     if ( ((p) = malloc(size)) == NULL)  { \\\n",
            "      |                 ^~~~~~~~~~~~\n",
            "hmmscan.c:402:3: note: in expansion of macro ‘ESL_ALLOC’\n",
            "  402 |   ESL_ALLOC(info, sizeof(*info) * infocnt);\n",
            "      |   ^~~~~~~~~\n",
            "In file included from hmmscan.c:11:\n",
            "hmmscan.c: In function ‘main’:\n",
            "/usr/include/stdlib.h:540:14: note: in a call to allocation function ‘malloc’ declared here\n",
            "  540 | extern void *malloc (size_t __size) __THROW __attribute_malloc__\n",
            "      |              ^~~~~~\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!wget http://ftp.ebi.ac.uk/pub/databases/Pfam/releases/Pfam35.0/Pfam-A.hmm.gz"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "7yyumuj2JQzf",
        "outputId": "7137526a-9bd5-4535-e121-7b8fcf478c0b"
      },
      "execution_count": 4,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2024-06-19 19:23:55--  http://ftp.ebi.ac.uk/pub/databases/Pfam/releases/Pfam35.0/Pfam-A.hmm.gz\n",
            "Resolving ftp.ebi.ac.uk (ftp.ebi.ac.uk)... 193.62.193.165\n",
            "Connecting to ftp.ebi.ac.uk (ftp.ebi.ac.uk)|193.62.193.165|:80... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 293000230 (279M) [application/x-gzip]\n",
            "Saving to: ‘Pfam-A.hmm.gz’\n",
            "\n",
            "Pfam-A.hmm.gz       100%[===================>] 279.43M   803KB/s    in 6m 3s   \n",
            "\n",
            "2024-06-19 19:29:58 (789 KB/s) - ‘Pfam-A.hmm.gz’ saved [293000230/293000230]\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!gunzip Pfam-A.hmm.gz"
      ],
      "metadata": {
        "id": "4SHs0rfCJSGz"
      },
      "execution_count": 5,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!hmmpress Pfam-A.hmm"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "47wNmAb0JTdx",
        "outputId": "0d7e8cf2-a5f6-46c8-f9dc-a6bbd2f9ff4b"
      },
      "execution_count": 6,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Working...    done.\n",
            "Pressed and indexed 19632 HMMs (19632 names and 19632 accessions).\n",
            "Models pressed into binary file:   Pfam-A.hmm.h3m\n",
            "SSI index for binary model file:   Pfam-A.hmm.h3i\n",
            "Profiles (MSV part) pressed into:  Pfam-A.hmm.h3f\n",
            "Profiles (remainder) pressed into: Pfam-A.hmm.h3p\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!hmmfetch --index Pfam-A.hmm.h3m"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "kisbjN4IJV9Z",
        "outputId": "428491cc-3e86-44a2-e517-fc8dc788551f"
      },
      "execution_count": 7,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Working...    done.\n",
            "Indexed 19632 HMMs (19632 names and 19632 accessions).\n",
            "SSI index written to file Pfam-A.hmm.h3m.ssi\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Формируем файлы с генами, отвечающими за эпигенетику"
      ],
      "metadata": {
        "id": "mJQXhGC8KGw0"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "\n",
        "Для работы были выбраны следующие `Pfam domains`:\n",
        "\n",
        "```\n",
        "PF09745\n",
        "PF01189\n",
        "PF05891\n",
        "PF04146\n",
        "PF00642\n",
        "PF17098\n",
        "PF15912\n",
        "PF01509\n",
        "PF15440\n",
        "PF15230\n",
        "```\n"
      ],
      "metadata": {
        "id": "dZSTLSNe6Ffp"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!grep PF09745 Pfam-A.hmm\n",
        "!grep PF01189 Pfam-A.hmm\n",
        "!grep PF05891 Pfam-A.hmm\n",
        "!grep PF04146 Pfam-A.hmm\n",
        "!grep PF00642 Pfam-A.hmm\n",
        "!grep PF17098 Pfam-A.hmm\n",
        "!grep PF15912 Pfam-A.hmm\n",
        "!grep PF01509 Pfam-A.hmm\n",
        "!grep PF15440 Pfam-A.hmm\n",
        "!grep PF15230 Pfam-A.hmm"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "niYdYz_QQIlQ",
        "outputId": "99e72962-a675-457b-de90-872a56597733"
      },
      "execution_count": 8,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "ACC   PF09745.12\n",
            "ACC   PF01189.20\n",
            "ACC   PF05891.15\n",
            "ACC   PF04146.18\n",
            "ACC   PF00642.27\n",
            "ACC   PF17098.8\n",
            "ACC   PF15912.8\n",
            "ACC   PF01509.21\n",
            "ACC   PF15440.9\n",
            "ACC   PF15230.9\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!echo -e \"PF09745.12\" > keyfile.txt\n",
        "!echo -e \"PF01189.20\" >> keyfile.txt\n",
        "!echo -e \"PF05891.15\" >> keyfile.txt\n",
        "!echo -e \"PF04146.18\" >> keyfile.txt\n",
        "!echo -e \"PF00642.27\" >> keyfile.txt\n",
        "!echo -e \"PF17098.8\" >> keyfile.txt\n",
        "!echo -e \"PF15912.8\" >> keyfile.txt\n",
        "!echo -e \"PF01509.21\" >> keyfile.txt\n",
        "!echo -e \"PF15440.9\" >> keyfile.txt\n",
        "!echo -e \"PF15230.9\" >> keyfile.txt"
      ],
      "metadata": {
        "id": "JZnsMQhaRFPm"
      },
      "execution_count": 9,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!cat keyfile.txt"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "0p8gQ-cf49zq",
        "outputId": "c217a386-4825-4764-cbf0-20c8fa0024f0"
      },
      "execution_count": 10,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PF09745.12\n",
            "PF01189.20\n",
            "PF05891.15\n",
            "PF04146.18\n",
            "PF00642.27\n",
            "PF17098.8\n",
            "PF15912.8\n",
            "PF01509.21\n",
            "PF15440.9\n",
            "PF15230.9\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!hmmfetch -f Pfam-A.hmm.h3m  keyfile.txt > Z-alpha.hmm"
      ],
      "metadata": {
        "id": "uhIw4PsPJZYD"
      },
      "execution_count": 11,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!hmmstat Z-alpha.hmm"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "sbX_gjgSQq4-",
        "outputId": "0ec96a6e-5832-4f6e-9fd0-2230eb8a3181"
      },
      "execution_count": 12,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "# hmmstat :: display summary statistics for a profile file\n",
            "# HMMER 3.3.2 (Nov 2020); http://hmmer.org/\n",
            "# Copyright (C) 2020 Howard Hughes Medical Institute.\n",
            "# Freely distributed under the BSD open source license.\n",
            "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
            "#\n",
            "# idx  name                 accession        nseq eff_nseq      M relent   info p relE compKL\n",
            "# ---- -------------------- ------------ -------- -------- ------ ------ ------ ------ ------\n",
            "1      NSRP1_N              PF09745.12        120     1.97    121   0.59   0.57   0.54   0.13\n",
            "2      Methyltr_RsmB-F      PF01189.20          8     1.11    200   0.59   0.59   0.52   0.01\n",
            "3      Methyltransf_PK      PF05891.15          7     0.80    218   0.59   0.58   0.52   0.01\n",
            "4      YTH                  PF04146.18        210     3.46    183   0.59   0.55   0.40   0.04\n",
            "5      zf-CCCH              PF00642.27         70    70.00     27   1.83   1.51   1.82   0.38\n",
            "6      Wtap                 PF17098.8          35     3.44    155   0.59   0.62   0.52   0.07\n",
            "7      VIR_N                PF15912.8           8     0.90    265   0.59   0.58   0.50   0.02\n",
            "8      TruB_N               PF01509.21        161     2.27    149   0.59   0.60   0.53   0.02\n",
            "9      THRAP3_BCLAF1        PF15440.9          25     2.33    662   0.59   0.57   0.47   0.10\n",
            "10     SRRM_C               PF15230.9          11     1.33     67   0.84   0.81   0.77   0.20\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Теперь найдем эти данные в файле с белками:"
      ],
      "metadata": {
        "id": "LhYCTTmrKbP1"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "Предварительно загрузим этот файл, скачанный с `ncbi`."
      ],
      "metadata": {
        "id": "XKfaeVQq_5_q"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!cat protein.faa > proteins.fasta"
      ],
      "metadata": {
        "id": "HM8jl3vkJza3"
      },
      "execution_count": 13,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!head proteins.fasta"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "dPMNUJe4_Q0v",
        "outputId": "6a2ed237-701a-4f87-ce75-9df3b36e5280"
      },
      "execution_count": 14,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            ">XP_002894699.1 secreted RxLR effector peptide protein, putative [Phytophthora infestans T30-4]\n",
            "MRLYLILSASMVISYFVTCNAATSSDQTKLSRMEPSLLVHSLGAALNVDAVDRRFLRERKESMEERGYHLTLKGEVKALA\n",
            "KKIIADFDTADDVYKKWNENGHSLNKIANLLKVSEKEKYVPVYNGYLAYLNRINS\n",
            ">XP_002894700.1 glycine-rich protein [Phytophthora infestans T30-4]\n",
            "MFSSSKITAVCLAVVALTNSAYAEKEAAQTFGLLGAGLHGGAGLYGAGAAGLHGGAGVGAGLYGRGAGYGGAGAGLYGRG\n",
            "AGGAGAGLYGRGAGYGGAGAGLYGRGAGYGGAGAGLYAGYGGAGAGLYGRGAGGVGAGLYGRGAGYGGAGAGLFGREAGY\n",
            "GGAGAGVGAGVGGAGLGNGDVIILLDDVTRNIIAKHISNKNLNCLVYNATKHPEVITLLYSSQVLGPECSHLL\n",
            ">XP_002894701.1 uncharacterized protein PITG_22119 [Phytophthora infestans T30-4]\n",
            "MTWSTEDLSEECTLARSQLQRAKGATKGWKESNGFNGHTVRTEIADGEPSQPVALDGRKSKDYRETENK\n",
            ">XP_002894702.1 arginase, putative [Phytophthora infestans T30-4]\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!hmmsearch --tblout hmmer_res.txt Z-alpha.hmm proteins.fasta"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "za_ZVjDlToxz",
        "outputId": "a09e345a-b273-4909-cc56-a57386453a2d"
      },
      "execution_count": 15,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "# hmmsearch :: search profile(s) against a sequence database\n",
            "# HMMER 3.3.2 (Nov 2020); http://hmmer.org/\n",
            "# Copyright (C) 2020 Howard Hughes Medical Institute.\n",
            "# Freely distributed under the BSD open source license.\n",
            "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
            "# query HMM file:                  Z-alpha.hmm\n",
            "# target sequence database:        proteins.fasta\n",
            "# per-seq hits tabular output:     hmmer_res.txt\n",
            "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
            "\n",
            "Query:       NSRP1_N  [M=121]\n",
            "Accession:   PF09745.12\n",
            "Description: Nuclear speckle splicing regulatory protein 1, N-terminal\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    4.1e-33  114.0  19.2    7.1e-33  113.2  19.2    1.4  1  XP_002895687.1  uncharacterized protein PITG_20187 [Phytophth\n",
            "  ------ inclusion threshold ------\n",
            "       0.53   10.3   5.0        7.9    6.5   0.2    3.1  3  XP_002999037.1  uncharacterized protein PITG_05382 [Phytophth\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_002895687.1  uncharacterized protein PITG_20187 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  113.2  19.2     8e-37   7.1e-33       3     121 .]      62     182 ..      60     182 .. 0.91\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 113.2 bits;  conditional E-value: 8e-37\n",
            "         NSRP1_N   3 keaekaleeDpsvydYDevYDslkkakekkkakkkeek..kerkpkYiesllkaaekRkrereraeerklqkereaegdefadkekFVTsaYkkk 95 \n",
            "                     k  ++al+eD+sv+dYD+vYD++k+a+e++++k+++++  +++kpkYi++ll++a+ R+ e+er++er+l  er+a++ ++ dkek V+++Yk+k\n",
            "  XP_002895687.1  62 KVRAQALAEDASVFDYDAVYDDMKSARESEADKRATKReeEKKKPKYISTLLQQAKIREVENERIRERRLLNERKADDVMYGDKEKLVSASYKRK 156\n",
            "                     56789*********************887766544444227889*************************************************** PP\n",
            "\n",
            "         NSRP1_N  96 leelrkleeeekkreeeeekkdvtkk 121\n",
            "                     l+e++++++e+++ ++ ee++dvtk+\n",
            "  XP_002895687.1 157 LQEMQRWDAEDARLAAMEEQEDVTKQ 182\n",
            "                     ***********************996 PP\n",
            "\n",
            ">> XP_002999037.1  uncharacterized protein PITG_05382 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.1   0.0     0.085   7.6e+02       5      48 ..       9      49 ..       5      60 .. 0.57\n",
            "   2 ?    3.7   0.2    0.0067        60      53      89 ..     173     209 ..     158     212 .. 0.89\n",
            "   3 ?    6.5   0.2   0.00088       7.9      68     100 ..     247     279 ..     243     286 .. 0.88\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.1 bits;  conditional E-value: 0.085\n",
            "         NSRP1_N  5 aekaleeDpsvydYDevYDslkkakekkkakkkeekkerkpkYi 48\n",
            "                      +++  D++v d+ +v D ++++++++ +++  e+++r+  ++\n",
            "  XP_002999037.1  9 KTTNSFVDDTVMDFNAV-DPTQARRRSEGSEQ--ESRNRHESWM 49\n",
            "                    55566678888888554.77666555444333..3334544455 PP\n",
            "\n",
            "  == domain 2  score: 3.7 bits;  conditional E-value: 0.0067\n",
            "         NSRP1_N  53 kaaekRkrereraeerklqkereaegdefadkekFVT 89 \n",
            "                     +++++R++ r+ +    +qk r +e+d  +d  k +T\n",
            "  XP_002999037.1 173 RHSQQRNNARRQRSSTVVQKVRPRENDDDEDDVKMIT 209\n",
            "                     567899999999999************9999999998 PP\n",
            "\n",
            "  == domain 3  score: 6.5 bits;  conditional E-value: 0.00088\n",
            "         NSRP1_N  68 rklqkereaegdefadkekFVTsaYkkkleelr 100\n",
            "                     +++  ere+ ++ f+d+ +  ++a++k ++  +\n",
            "  XP_002999037.1 247 KTVTTERERWENDFEDEMRVLSAAFRKAHASEK 279\n",
            "                     78899*********************9987765 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (121 nodes)\n",
            "Target sequences:                      17797  (7477560 residues searched)\n",
            "Passed MSV filter:                      2117  (0.118953); expected 355.9 (0.02)\n",
            "Passed bias filter:                      479  (0.0269146); expected 355.9 (0.02)\n",
            "Passed Vit filter:                        47  (0.00264089); expected 17.8 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000112378); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              17797  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.21u 0.01s 00:00:00.22 Elapsed: 00:00:00.24\n",
            "# Mc/sec: 3639.89\n",
            "//\n",
            "Query:       Methyltr_RsmB-F  [M=200]\n",
            "Accession:   PF01189.20\n",
            "Description: 16S rRNA methyltransferase RsmB/F\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    1.7e-74  249.8   0.0    3.4e-74  248.8   0.0    1.5  1  XP_002907645.1  ribosomal RNA methyltransferase nop2-like pro\n",
            "    3.9e-38  131.1   0.0    1.6e-35  122.5   0.0    2.3  1  XP_002898009.1  uncharacterized protein PITG_15740 [Phytophth\n",
            "    2.9e-37  128.3   0.0    2.8e-36  125.0   0.0    2.2  1  XP_002896768.1  tRNA (cytosine-5-)-methyltransferase NSUN2-li\n",
            "    6.9e-34  117.2   0.0    1.6e-31  109.5   0.0    2.3  2  XP_002909047.1  tRNA (cytosine-5-)-methyltransferase NSUN2, p\n",
            "      2e-31  109.2   0.0    4.6e-29  101.5   0.0    2.3  2  XP_002896806.1  tRNA (cytosine-5-)-methyltransferase, putativ\n",
            "    4.6e-28   98.2   0.3    1.6e-15   57.3   0.2    3.0  3  XP_002896631.1  uncharacterized protein PITG_17784 [Phytophth\n",
            "    7.3e-19   68.2   0.0    2.9e-18   66.2   0.0    2.1  1  XP_002907476.1  tRNA (cytosine-5-)-methyltransferase NSUN2-li\n",
            "    2.2e-16   60.1   0.7    1.2e-14   54.4   0.7    2.5  1  XP_002998084.1  uncharacterized protein PITG_05843 [Phytophth\n",
            "      1e-07   31.8   0.0    1.6e-07   31.2   0.0    1.2  1  XP_002908463.1  uncharacterized protein PITG_01864 [Phytophth\n",
            "     0.0013   18.5   0.0     0.0021   17.7   0.0    1.4  1  XP_002897399.1  ribosomal RNA methyltransferase, putative [Ph\n",
            "     0.0044   16.7   0.0      0.015   15.0   0.0    1.8  2  XP_002998926.1  tRNA (adenine-N(1)-)-methyltransferase cataly\n",
            "     0.0054   16.4   0.0     0.0096   15.6   0.0    1.3  1  XP_002904831.1  ribosomal RNA methyltransferase, putative [Ph\n",
            "  ------ inclusion threshold ------\n",
            "       0.12   12.0   0.0       0.19   11.3   0.0    1.3  1  XP_002901056.1  protein-L-isoaspartate O-methyltransferase [P\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_002907645.1  ribosomal RNA methyltransferase nop2-like protein [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  248.8   0.0   2.5e-77   3.4e-74       1     200 []     306     514 ..     306     514 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 248.8 bits;  conditional E-value: 2.5e-77\n",
            "                      HHHH---TT-EEEESS-TTSHHHHHHHHHTTTTSEEEEEESSCCCHHHHHHHHHHTT--SEEEEES-CTCCHHHHCC..EEEEEEEE--CCGGG CS\n",
            "  Methyltr_RsmB-F   1 allLapqeeetilDlcaaPGgKTthiaelmknkgtvvAvdankeRlkrvaenlkrLgvtntivleaDarkidkvlggelfDriLlDaPCsgtgv 94 \n",
            "                      +++Lapq +e++lD+++aPGgKTt+ia+lmkn+gtv+A+d +k+Rlk+ ++nl+rLgv+nt v+++D+rk+++   g  fDr+LlDaPC+g+gv\n",
            "   XP_002907645.1 306 VMALAPQMNERVLDMACAPGGKTTYIAQLMKNTGTVIANDLKKQRLKATVANLHRLGVKNTSVSCYDGRKVPSLFKG--FDRVLLDAPCTGMGV 397\n",
            "                      589*************************************************************************9..*************** PP\n",
            "\n",
            "                      TTT-TCHHHH--HHHHHHHHHHHHHHHHHHCCCE....EEEEEEEEEES--STTTTHHHHHHHHHH-TTEEEESSBSTTSTTSSECEEC..... CS\n",
            "  Methyltr_RsmB-F  95 irrdpdvklrrkekdiaqlaklqkellsaaidll....kkgGvlvYsTCsvlveEneevieeaLkkredvelvetelsepkglklgeea..... 179\n",
            "                      i rdp++k+++ ekd+++la+lqkell aaid +    k+gG+++YsTCsv v+Ene v+++aL+kr  v+lv+t l ++k   +  ++     \n",
            "   XP_002907645.1 398 IARDPSIKTQKGEKDVQRLAHLQKELLLAAIDAVdaksKTGGYILYSTCSVMVDENESVVDYALRKR-CVKLVDTGLDHGKPGFTRYQQqrfhp 490\n",
            "                      *******************************************************************.99*****9887653322222246999 PP\n",
            "\n",
            "                      ...TSEEE-TTTTSS-SEEEEEEE CS\n",
            "  Methyltr_RsmB-F 180 ...gtlrllPhtenkdgffiaklk 200\n",
            "                          t+r++Ph +n+dgf++ak+k\n",
            "   XP_002907645.1 491 svqLTRRFYPHVHNMDGFYVAKFK 514\n",
            "                      9999******************96 PP\n",
            "\n",
            ">> XP_002898009.1  uncharacterized protein PITG_15740 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  122.5   0.0   1.2e-38   1.6e-35       2     199 ..     240     480 ..     239     481 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 122.5 bits;  conditional E-value: 1.2e-38\n",
            "                      HHH---TT-EEEESS-TTSHHHHHHHHHTTTTSEEEEEESSCCCHHHHHHHHHHTT--SEEEEES-CTCC..HHHHCC................ CS\n",
            "  Methyltr_RsmB-F   2 llLapqeeetilDlcaaPGgKTthiaelmknkgtvvAvdankeRlkrvaenlkrLgvtntivleaDarki..dkvlgg................ 77 \n",
            "                       +L+pq+++ ilD+caaPGgKT+h+a lm+n+gt+vA d++k+++ ++ +  + L ++ +  l++D ++   +k+  +                \n",
            "   XP_002898009.1 240 HVLDPQQGDVILDMCAAPGGKTSHVATLMRNTGTLVACDRSKRKALELRTLCEDLHLECVEPLKMDSTHAllPKD--Kvdaaprdgdftsvaqv 331\n",
            "                      68********************************************************99999999998622222..2335788999******* PP\n",
            "\n",
            "                      ........................EEEEEEEE--CCGGGTTT-TCHHHH--HHHHHHHHHHHHHHHHHHCCCEEEEEEEEEEES--STTTTHHH CS\n",
            "  Methyltr_RsmB-F  78 ......................elfDriLlDaPCsgtgvirrdpdvklrrkekdiaqlaklqkellsaaidllkkgGvlvYsTCsvlveEneev 149\n",
            "                                            e+fDriLlD PCs++g   r        +  ++a++++ q++ l  a+ llk+gG+lvYsTC+++++Ene++\n",
            "   XP_002898009.1 332 larakqstprqtrllqvegfylETFDRILLDPPCSALGLRPRLL---HAGDADNLAEYTNMQRNFLWVAAFLLKPGGTLVYSTCTINPKENEQM 422\n",
            "                      **************************************966654...4588899**************************************** PP\n",
            "\n",
            "                      HHHHHHH-TTEEEESSBSTTSTTSSECEEC......TSEEE-TTTTSS-..SEEEEEE CS\n",
            "  Methyltr_RsmB-F 150 ieeaLkkredvelvetelsepkglklgeea......gtlrllPhtenkd..gffiakl 199\n",
            "                      +++aL++ +   +++ +++ +++   g++         +r+ P + + d  gff ak+\n",
            "   XP_002898009.1 423 VHHALQNYPLKLVSQGDAHLGDHGLSGQGLneeevaLVQRFDPANTELDtmGFFCAKF 480\n",
            "                      ******9864433333333222211111113333548889999999888667999997 PP\n",
            "\n",
            ">> XP_002896768.1  tRNA (cytosine-5-)-methyltransferase NSUN2-like protein [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  125.0   0.0     2e-39   2.8e-36       2     199 ..     149     417 ..     148     418 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 125.0 bits;  conditional E-value: 2e-39\n",
            "                      HHH---TT-EEEESS-TTSHHHHHHHH......H.TTTTSEEEEEESSCCCHHHHHHHHHHTT--SEEEEES-CTCCHHHHCC.........EE CS\n",
            "  Methyltr_RsmB-F   2 llLapqeeetilDlcaaPGgKTthiae......l.mknkgtvvAvdankeRlkrvaenlkrLgvtntivleaDarkidkvlgg.......elfD 81 \n",
            "                      llL+ +++ ++lD+caaPG+KTt i+e      + ++++g vvA+dan++R   +++ l+rLg+ n  v++ +++k++            ++fD\n",
            "   XP_002896768.1 149 LLLDVHPHHHVLDMCAAPGSKTTQIMEalisdkVsINTQGFVVANDANEKRGYLLVHQLQRLGLDNFVVTCHEGQKFPGLYNSnvelqrtNVFD 242\n",
            "                      7899********************98732222223467899************************************98777678899999*** PP\n",
            "\n",
            "                      EEEEEE--CCGGGTTT-TCHHHH--HHHHHHHHHHHHHHHHHHCCCEEEEE..EEEEEES--STTTTHHHHHHHHHH-TTEEEESSBSTT...S CS\n",
            "  Methyltr_RsmB-F  82 riLlDaPCsgtgvirrdpdvklrrkekdiaqlaklqkellsaaidllkkgG..vlvYsTCsvlveEneevieeaLkkredvelvetelse...p 170\n",
            "                      r+L D+PCsg g+ir+++++  r  +    +l+  q +l   a++ll+ +G  ++ YsTCs+++ Ene+v++e+L++  d +l+ +++se   +\n",
            "   XP_002896768.1 243 RVLCDVPCSGDGTIRKNRNLWGRWAPGSALTLHPIQIDLGLRAAALLRDDGdsIMTYSTCSLNPVENEAVVAELLRRA-DGALELVDCSEklsG 335\n",
            "                      ********************88889***********************99955688*******************987.455555544540013 PP\n",
            "\n",
            "                      TTSSECEEC.....................................................TSEEE-TTTTSS-SEEEEEE CS\n",
            "  Methyltr_RsmB-F 171 kglklgeea.....................................................gtlrllPhtenkdgffiakl 199\n",
            "                       +++ g+++                                                       lrl+P  +n+ gff+a l\n",
            "   XP_002896768.1 336 LITRPGVTYwsvawqartakgearaplqwfdqyesvpdflqgsrilrsmfppadeeiqkilpRCLRLFPTDQNTGGFFVAVL 417\n",
            "                      4555666669999999999999999999999999999999999999999999999999999989999999999999999987 PP\n",
            "\n",
            ">> XP_002909047.1  tRNA (cytosine-5-)-methyltransferase NSUN2, putative [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  109.5   0.0   1.2e-34   1.6e-31       3     157 ..     168     336 ..     166     356 .. 0.85\n",
            "   2 !    5.4   0.0    0.0094        13     151     199 ..     379     428 ..     366     429 .. 0.61\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 109.5 bits;  conditional E-value: 1.2e-34\n",
            "                      HH---TT-EEEESS-TTSHHHHHHHHHTT.........TTSEEEEEESSCCCHHHHHHHHHHTT--SEEEEES-CTCCHHHHCC........EE CS\n",
            "  Methyltr_RsmB-F   3 lLapqeeetilDlcaaPGgKTthiaelmk.........nkgtvvAvdankeRlkrvaenlkrLgvtntivleaDarkidkvlgg......elfD 81 \n",
            "                      +L  + + ++lD+caaPG+KT+ ++e ++         ++g+vvA+d++ +R+  +++  kr++   + v++ +a++i+  lg+       +fD\n",
            "   XP_002909047.1 168 VLGVESHHKVLDMCAAPGSKTSQLLESLHsqefatgktPTGVVVANDVDLKRAYMLVHQSKRISSPALLVTCHEAQNIPF-LGKdgteseGVFD 260\n",
            "                      677889999*************99986541112222225789************************99999999999964.33346667689** PP\n",
            "\n",
            "                      EEEEEE--CCGGGTTT-TCHHHH--HHHHHHHHHHHHHHHHHHCCCEEEEEEEEEEES--STTTTHHHHHHHHHH- CS\n",
            "  Methyltr_RsmB-F  82 riLlDaPCsgtgvirrdpdvklrrkekdiaqlaklqkellsaaidllkkgGvlvYsTCsvlveEneevieeaLkkr 157\n",
            "                      riL DaPCsg g++r++p +  + + k+ ++l+ lq ++ +  ++llk gG + YsTC+ ++ Ene+v++++L+  \n",
            "   XP_002909047.1 261 RILCDAPCSGDGTLRKNPLIWKNWSAKNGIALHPLQLQIAKRGAALLKVGGNMCYSTCTFNPLENEAVVADLLRWS 336\n",
            "                      ********************8899999**********************************************954 PP\n",
            "\n",
            "  == domain 2  score: 5.4 bits;  conditional E-value: 0.0094\n",
            "                      HHHHHH-TTEEEESS..BSTTSTTSSECEECTSEEE-TTTTSS-SEEEEEE CS\n",
            "  Methyltr_RsmB-F 151 eeaLkkredvelvet..elsepkglklgeeagtlrllPhtenkdgffiakl 199\n",
            "                      ee +k++ + ++  t  + ++ + ++l  e   +r lP+ en+ gffi  l\n",
            "   XP_002909047.1 379 EENMKDKIKDKIRATmfSPTKKEAADLHLER-CMRCLPQDENTGGFFICLL 428\n",
            "                      4444444333333330022333344455555.89**************987 PP\n",
            "\n",
            ">> XP_002896806.1  tRNA (cytosine-5-)-methyltransferase, putative [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  101.5   0.0   3.3e-32   4.6e-29       2     157 ..     147     319 ..     146     341 .. 0.86\n",
            "   2 !    5.4   0.0    0.0092        13     173     200 .]     401     428 ..     367     428 .. 0.77\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 101.5 bits;  conditional E-value: 3.3e-32\n",
            "                      HHH---TT-EEEESS-TTSHHHHHHHHHT.........TTTSEEEEEESSCCCHHHHHHHHHHTT--SEEEEES-CTCCHHHHCC......... CS\n",
            "  Methyltr_RsmB-F   2 llLapqeeetilDlcaaPGgKTthiaelm.........knkgtvvAvdankeRlkrvaenlkrLgvtntivleaDarkidkvlgg........e 78 \n",
            "                      l+L+ q   ++lD+ca+PG+KTt + +++         +++g v+A+d +k+R+  +++ l+r  ++ + v+++ +  ++             +\n",
            "   XP_002896806.1 147 LFLDVQRSHRVLDMCASPGSKTTQVIDFLlstengdngEQSGLVIANDLDKKRAYMLVHRLSRNTLRRAVVTCGAGDTFPGLYDAktktlqptN 240\n",
            "                      6799999****************99887423333333234678************************999999988888765544567788889 PP\n",
            "\n",
            "                      .EEEEEEEE--CCGGGTTT-TCHHHH--HHHHHHHHHHHHHHHHHHCCCEEEEEEEEEEES--STTTTHHHHHHHHHH- CS\n",
            "  Methyltr_RsmB-F  79 lfDriLlDaPCsgtgvirrdpdvklrrkekdiaqlaklqkellsaaidllkkgGvlvYsTCsvlveEneevieeaLkkr 157\n",
            "                      +fDr+L D+PCsg g++r+++ +  + +  +  +l+  q +l    ++ll+ +G++vYsTCs ++ Ene+v++e+L++ \n",
            "   XP_002896806.1 241 VFDRVLCDVPCSGDGTLRKNQTLWKEWHIGQGLTLHPTQLALALRGAALLRVNGIMVYSTCSFNPVENEAVVAELLRRA 319\n",
            "                      ***********************66667788899******************************************985 PP\n",
            "\n",
            "  == domain 2  score: 5.4 bits;  conditional E-value: 0.0092\n",
            "                      SSECEECTSEEE-TTTTSS-SEEEEEEE CS\n",
            "  Methyltr_RsmB-F 173 lklgeeagtlrllPhtenkdgffiaklk 200\n",
            "                      +   e ++tlrl+P  +n  gffia l+\n",
            "   XP_002896806.1 401 VIADELHKTLRLIPTDQNSGGFFIAVLR 428\n",
            "                      4444445799***************985 PP\n",
            "\n",
            ">> XP_002896631.1  uncharacterized protein PITG_17784 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.3   0.0      0.49   6.8e+02      31      62 ..      50      81 ..      46      95 .. 0.79\n",
            "   2 !   37.0   0.0     2e-12   2.7e-09       2      55 ..     130     183 ..     129     208 .. 0.91\n",
            "   3 !   57.3   0.2   1.2e-18   1.6e-15      66     169 ..     251     365 ..     232     379 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.3 bits;  conditional E-value: 0.49\n",
            "                     TTTSEEEEEESSCCCHHHHHHHHHHTT--SEE CS\n",
            "  Methyltr_RsmB-F 31 knkgtvvAvdankeRlkrvaenlkrLgvtnti 62\n",
            "                     k+ + +   d+++ Rl+++a+nl r   +n+ \n",
            "   XP_002896631.1 50 KQLNSISEKDTEDPRLQAIAANLLREAAENVS 81\n",
            "                     556678889999**********9998877665 PP\n",
            "\n",
            "  == domain 2  score: 37.0 bits;  conditional E-value: 2e-12\n",
            "                      HHH---TT-EEEESS-TTSHHHHHHHHHTTTTSEEEEEESSCCCHHHHHHHHHH CS\n",
            "  Methyltr_RsmB-F   2 llLapqeeetilDlcaaPGgKTthiaelmknkgtvvAvdankeRlkrvaenlkr 55 \n",
            "                      +lL+ +++e++lDlc+aPG+K t ia+l++ +g v+ vd +++Rl +  + +++\n",
            "   XP_002896631.1 130 TLLNINPGEHVLDLCCAPGAKLTMIADLLQLRGSVTGVDFSRSRLGACRQLVHK 183\n",
            "                      789*****************************************9887776665 PP\n",
            "\n",
            "  == domain 3  score: 57.3 bits;  conditional E-value: 1.2e-18\n",
            "                      S-CTCCHHHHCC...EEEEEEEE--CCGGGTTT-TCH....HHH-......-HHHHHHHHHHHHHHHHHHCCCEEEEEEEEEEES--STTTTHH CS\n",
            "  Methyltr_RsmB-F  66 aDarkidkvlgg.elfDriLlDaPCsgtgvirrdpdv....klrr......kekdiaqlaklqkellsaaidllkkgGvlvYsTCsvlveEnee 148\n",
            "                       Da++ +  + + +l+D++L+Da C+  g ir+ + +    k++       + ++i ++ klq+ l+   + ll++gG+++YsTCs+ v+ ne \n",
            "   XP_002896631.1 251 RDAKRQKLLVVDsALYDKVLVDAECTHDGSIRHLQRLitgaKWEEyvrdhlNSSEIERILKLQHGLIRNGFSLLRPGGTMIYSTCSLSVKQNED 344\n",
            "                      566665555554499***************9987653233366542212225678*************************************** PP\n",
            "\n",
            "                      HHHHHHHH-TTEEEESSBSTT CS\n",
            "  Methyltr_RsmB-F 149 vieeaLkkredvelvetelse 169\n",
            "                       + ++L++++ + l++t + +\n",
            "   XP_002896631.1 345 IVSKFLEDQPLATLDQTTVDH 365\n",
            "                      **************9997765 PP\n",
            "\n",
            ">> XP_002907476.1  tRNA (cytosine-5-)-methyltransferase NSUN2-like protein [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.2   0.0   2.1e-21   2.9e-18      19     199 ..     283     507 ..     252     508 .. 0.76\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.2 bits;  conditional E-value: 2.1e-21\n",
            "                      TSHHHHHHHHH.TTTTSEEEEEESSCCCHHHHHHHHHH.TT--.SEEEEES-CTCCHH.HHCC..EEEEEEEE--CCGGGTTT-TCHHHH--HH CS\n",
            "  Methyltr_RsmB-F  19 PGgKTthiael.mknkgtvvAvdankeRlkrvaenlkr.Lgvt.ntivleaDarkidk.vlggelfDriLlDaPCsgtgvirrdpdvklrrkek 108\n",
            "                      PG+  +   e   + k+++v ++++++ + + ++nl+r L++  ++iv+      ++   ++++lfDr++  aPCsg g ir+ p+   + +++\n",
            "   XP_002907476.1 283 PGSPASAGIETsSSAKNVLVVNERDATVAASAVRNLTRtLPLAcDLIVSAHKPEDFPVpGIADALFDRVICCAPCSGDGLIRKLPEKWRTWSPE 376\n",
            "                      6666655555513445556677888888888888877656776345555554444432135569**********************75556889 PP\n",
            "\n",
            "                      HHHHHHHHHHHHHHHHCCCEEEEEEEEEEES--STTTTHHHHHHHHHH-T.TEEEESSB.STTSTTSSECEEC..................... CS\n",
            "  Methyltr_RsmB-F 109 diaqlaklqkellsaaidllkkgGvlvYsTCsvlveEneevieeaLkkre.dvelvete.lsepkglklgeea..................... 179\n",
            "                      +    +  q  l + a+ ll++gG l+YsT s+++ Ene+v++e+L++ +   elvet+ + e+ + + g  +                     \n",
            "   XP_002907476.1 377 QALVHHPSQLGLAEHAVSLLRTGGALLYSTRSLNPIENEAVVAELLRRSNgALELVETDdILEGLERSHGLSQwdvldlaswddasedqrhrlr 470\n",
            "                      99999999**************************************99862456766632445555555555599******************* PP\n",
            "\n",
            "                      .................TSEEE-TTTTSS-SEEEEEE CS\n",
            "  Methyltr_RsmB-F 180 .................gtlrllPhtenkdgffiakl 199\n",
            "                                         +rllPh++++ g+f+a +\n",
            "   XP_002907476.1 471 psmwppsneecdemhldRCVRLLPHQNDTHGLFLAVI 507\n",
            "                      **************999999*************9965 PP\n",
            "\n",
            ">> XP_002998084.1  uncharacterized protein PITG_05843 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   54.4   0.7   8.9e-18   1.2e-14       8     160 ..     218     400 ..     212     450 .. 0.70\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 54.4 bits;  conditional E-value: 8.9e-18\n",
            "                      TT-EEEESS-TTSHHHHHHHHHTTTTS...........EEEEEESSCCCHHHHHHHHHHTT--SEEEEES-CTCCHHHHCC......EEEEEEE CS\n",
            "  Methyltr_RsmB-F   8 eeetilDlcaaPGgKTthiaelmknkg...........tvvAvdankeRlkrvaenlkrLgvtntivleaDarkidkvlgg....elfDriLlD 86 \n",
            "                      ++  ++D+caaPG KT+h+a l++++            +v+A d++ +Rl+ +++ ++  g    + ++++++ + +v  +    + +  iLlD\n",
            "   XP_002998084.1 218 NQGDVIDACAAPGNKTSHLAMLLQQQDgdgedatcpkrRVFAFDRSPKRLDLLKRRMTLAGAA--TRVQVELQSFLDVPVDddmyRNVRSILLD 309\n",
            "                      44459*****************88652222333333338**************9988877776..4456666666666666556667999**** PP\n",
            "\n",
            "                      E--CCGGGTTT-TCHH.HH--HHHH......HHHHHHHHHHHHHHCCCE..........EEEEEEEEEES--STTTTHHHHHHHHHH-TTE CS\n",
            "  Methyltr_RsmB-F  87 aPCsgtgvirrdpdvk.lrrkekdi......aqlaklqkellsaaidll..........kkgGvlvYsTCsvlveEneevieeaLkkredv 160\n",
            "                        Csg g+ +r  ++  +  +  ++      a++ +   + l++++d             +   +vYsTCs+  +E+eev+++aL+++e+ \n",
            "   XP_002998084.1 310 PSCSGSGMTNRLDHLLdIASSHDTVlepdhgAEYEEDTAKRLQSLADFQlqalrkafsfPQVKRVVYSTCSIFQKEDEEVVAAALESEENS 400\n",
            "                      ********999664431222222221111113444444444444443333444544454455679*********************99876 PP\n",
            "\n",
            ">> XP_002908463.1  uncharacterized protein PITG_01864 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   31.2   0.0   1.2e-10   1.6e-07       2     137 ..     121     256 ..     120     261 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 31.2 bits;  conditional E-value: 1.2e-10\n",
            "                      HHH---TT-EEEESS-TTSHHHHHHHHHTTTTSEEEEEESSCCCHHHHHHHHHHT..T--..SEEEEES-CTCCHHHHCC..EEEEEEEE--CC CS\n",
            "  Methyltr_RsmB-F   2 llLapqeeetilDlcaaPGgKTthiaelmknkgtvvAvdankeRlkrvaenlkrL..gvt..ntivleaDarkidkvlggelfDriLlDaPCsg 91 \n",
            "                      l+L p  e+ +lDlca+ G   + i+e   ++ ++v vd +++Rl    + l+ +  gv+     v+++ +    + l +e +D   + + C +\n",
            "   XP_002908463.1 121 LALPPCGEKAVLDLCAGSGRVSAAILEA-YPTVKLVLVDSSEQRLTLASQRLEAVqaGVKerTQFVTKVVTPTATTQLCDEPVD---IVVACLA 210\n",
            "                      678899999******************9.78888999*******9988877776522344212233333333333344445555...4578*** PP\n",
            "\n",
            "                      GGGTTT-TCHHHH--HHHHHHHHHHHHHHHHHHCCCEEEEEEEEEE CS\n",
            "  Methyltr_RsmB-F  92 tgvirrdpdvklrrkekdiaqlaklqkellsaaidllkkgGvlvYs 137\n",
            "                        v+  +p+     +e+  +++    k+l+ a++++l++gG ++Y+\n",
            "   XP_002908463.1 211 FHVLTEKPAHYAPAEEEKTMAVEDEYKQLFRAVWKTLRPGGHVIYA 256\n",
            "                      *********************************************8 PP\n",
            "\n",
            ">> XP_002897399.1  ribosomal RNA methyltransferase, putative [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.7   0.0   1.6e-06    0.0021      11      42 ..      44      75 ..      39     113 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.7 bits;  conditional E-value: 1.6e-06\n",
            "                     EEEESS-TTSHHHHHHHHHTTTTSEEEEEESS CS\n",
            "  Methyltr_RsmB-F 11 tilDlcaaPGgKTthiaelmknkgtvvAvdan 42\n",
            "                     +++DlcaaPGg    iae + + +t++Avd  \n",
            "   XP_002897399.1 44 RVVDLCAAPGGWSQVIAERVPKDSTIIAVDLM 75\n",
            "                     699**************************975 PP\n",
            "\n",
            ">> XP_002998926.1  tRNA (adenine-N(1)-)-methyltransferase catalytic subunit, putative [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.0   0.0   1.1e-05     0.015      11      89 ..     125     205 ..     116     219 .. 0.77\n",
            "   2 ?   -1.5   0.0       1.2   1.7e+03     124     150 ..     213     240 ..     208     255 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.0 bits;  conditional E-value: 1.1e-05\n",
            "                      EEEESS-TTSHHHHHHHHHTTTTSEEEEEESSCCCHHHHHHHHHHTT--SEEEEES-CTCCHHHHCC......EEEEEEEE-- CS\n",
            "  Methyltr_RsmB-F  11 tilDlcaaPGgKTthiaelmknkgtvvAvdankeRlkrvaenlkrLgvtntivleaDarkidkvlgg....elfDriLlDaPC 89 \n",
            "                      ++++  ++ G+ Tt  a  ++++g+v+  + n++R++   e  kr g++   v+++D r   +         ++D + lD P \n",
            "   XP_002998926.1 125 RVVESGTGSGALTTSFARTVAPHGHVFTFEFNAHRAEIAREEFKRNGLE--SVITVDCRDACEQGFPveleGTIDMVFLDLPS 205\n",
            "                      566677788999*************************************..45555555443221111233699*****9995 PP\n",
            "\n",
            "  == domain 2  score: -1.5 bits;  conditional E-value: 1.2\n",
            "                      HCCCEEEEEEEE.EEES--STTTTHHHH CS\n",
            "  Methyltr_RsmB-F 124 aidllkkgGvlv.YsTCsvlveEneevi 150\n",
            "                      a+++lk+gG+   Ys C  +v+ ++ ++\n",
            "   XP_002998926.1 213 AAKMLKQGGFFAsYSPCIEQVQKTCGAL 240\n",
            "                      6778999998744999998888877665 PP\n",
            "\n",
            ">> XP_002904831.1  ribosomal RNA methyltransferase, putative [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.6   0.0     7e-06    0.0096       7      69 ..      40     101 ..      38     115 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.6 bits;  conditional E-value: 7e-06\n",
            "                      -TT-EEEESS-TTSHHHHHHHHHTTTTSEEEEEESSCCCHHHHHHHHHHTT--SEEEEES-CT CS\n",
            "  Methyltr_RsmB-F   7 qeeetilDlcaaPGgKTthiaelmknkgtvvAvdankeRlkrvaenlkrLgvtntivleaDar 69 \n",
            "                      ++ ++ +DlcaaPG+    ++  + ++++v  vd ++ R+ +v   ++  ++  +++l++D++\n",
            "   XP_002904831.1  40 RNVQRAVDLCAAPGSWSQVLSRKLYDASNVQSVDSDDVRIMSVDL-QEMAPIAGVQLLQGDIT 101\n",
            "                      5678899***************99****************99853.33345556677777765 PP\n",
            "\n",
            ">> XP_002901056.1  protein-L-isoaspartate O-methyltransferase [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.3   0.0   0.00014      0.19       5      58 ..      66     119 ..      62     145 .. 0.78\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.3 bits;  conditional E-value: 0.00014\n",
            "                      ---TT-EEEESS-TTSHHHHHHHHHTTTTSEEEEEESSCCCHHHHHHHHHHTT- CS\n",
            "  Methyltr_RsmB-F   5 apqeeetilDlcaaPGgKTthiaelmknkgtvvAvdankeRlkrvaenlkrLgv 58 \n",
            "                      +  e  ++lD+ a+ G  T+ + +l+++ g+++ +++  + ++  ++n++r   \n",
            "   XP_002901056.1  66 HNIEYPRVLDVGAGSGYLTACLGHLVDEGGKIFGIERIPQLAQLAQQNIERADG 119\n",
            "                      5556668*******************************************9653 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (200 nodes)\n",
            "Target sequences:                      17797  (7477560 residues searched)\n",
            "Passed MSV filter:                       646  (0.0362983); expected 355.9 (0.02)\n",
            "Passed bias filter:                      450  (0.0252852); expected 355.9 (0.02)\n",
            "Passed Vit filter:                        54  (0.00303422); expected 17.8 (0.001)\n",
            "Passed Fwd filter:                        13  (0.00073046); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              17797  [actual number of targets]\n",
            "Domain search space  (domZ):              13  [number of targets reported over threshold]\n",
            "# CPU time: 0.28u 0.01s 00:00:00.29 Elapsed: 00:00:00.35\n",
            "# Mc/sec: 4239.29\n",
            "//\n",
            "Query:       Methyltransf_PK  [M=218]\n",
            "Accession:   PF05891.15\n",
            "Description: AdoMet dependent proline di-methyltransferase\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    1.2e-72  244.0   0.0    1.3e-72  243.8   0.0    1.0  1  XP_002997291.1  uncharacterized protein PITG_18863 [Phytophth\n",
            "    2.7e-12   46.7   0.1    1.6e-10   40.8   0.0    2.6  2  XP_002898137.1  phosphoethanolamine N-methyltransferase, puta\n",
            "    4.9e-11   42.5   0.0    3.3e-09   36.6   0.0    2.6  2  XP_002898136.1  phosphoethanolamine N-methyltransferase [Phyt\n",
            "    0.00069   19.2   0.0     0.0011   18.5   0.0    1.3  1  XP_002908111.1  uncharacterized protein PITG_01446 [Phytophth\n",
            "  ------ inclusion threshold ------\n",
            "      0.026   14.0   0.0      0.071   12.6   0.0    1.6  1  XP_002905021.1  ribosomal protein S6 kinase, putative [Phytop\n",
            "      0.029   13.9   0.0      0.047   13.2   0.0    1.2  1  XP_002905459.1  uncharacterized protein PITG_04729 [Phytophth\n",
            "      0.029   13.9   0.0      0.047   13.2   0.0    1.3  1  XP_002905473.1  uncharacterized protein PITG_04747 [Phytophth\n",
            "      0.029   13.9   0.0      0.047   13.2   0.0    1.3  1  XP_002900712.1  uncharacterized protein PITG_12077 [Phytophth\n",
            "      0.089   12.3   0.0        8.5    5.8   0.0    2.1  2  XP_002905524.1  uncharacterized protein PITG_04818 [Phytophth\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_002997291.1  uncharacterized protein PITG_18863 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  243.8   0.0   6.7e-76   1.3e-72       4     217 ..      28     247 ..      25     248 .] 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 243.8 bits;  conditional E-value: 6.7e-76\n",
            "  Methyltransf_PK   4 evyekaiaYWes...veadvdGvLGGygavskiDvkgsrkflkklkkeslpgkrevkvaldcGaGiGrvtknlLlkllskvDlvepvekfleka 94 \n",
            "                      ++y  a +YW++      + +GvLGG+ +vs +D+  sr+flk+++      +  ++ a+dcGaGiGrv+k lLl+++ +vD+ve   ++l  +\n",
            "   XP_002997291.1  28 TWYASAHNYWDDeanCPLTDNGVLGGFAHVSGVDIRESRRFLKHVRDTVR-PEWVCHAAVDCGAGIGRVSKLLLLPMFDHVDMVEQSPRLLRGV 120\n",
            "                      79********8622256799*************************99955.6778999************************************ PP\n",
            "\n",
            "  Methyltransf_PK  95 keelaeekk...kvgkvyvvglqeftPeegrYdliWiqWcvghLtDedlvaflkrckkalkpnGlivvkenvake.ekdelDkeDssvtrseak 184\n",
            "                      +++l+ +++   +v+++y++glq+f P+  +YdliW+qW++ hLtD dlv +lkrckkal pnG+iv+kenv ++ e  +lD +Dss+trs  +\n",
            "   XP_002997291.1 121 SQYLGADETlcvRVRDLYCMGLQDFEPAPASYDLIWMQWVLVHLTDLDLVRYLKRCKKALTPNGFIVIKENVFQTpEPYDLDRQDSSITRSAVY 214\n",
            "                      *****966434499*********************************************************987635569************** PP\n",
            "\n",
            "  Methyltransf_PK 185 lkklfkkagLklvaeelqkglPkelypvkmyaL 217\n",
            "                      +k++f++agL+l+ae++q+ +P+elypv myaL\n",
            "   XP_002997291.1 215 YKSIFQQAGLTLLAEQRQRHFPEELYPVIMYAL 247\n",
            "                      ********************************9 PP\n",
            "\n",
            ">> XP_002898137.1  phosphoethanolamine N-methyltransferase, putative [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   40.8   0.0   8.3e-14   1.6e-10      53     166 ..      87     197 ..      49     206 .. 0.82\n",
            "   2 ?    2.9   0.0     0.033        66      42     161 ..     314     425 ..     279     437 .. 0.68\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 40.8 bits;  conditional E-value: 8.3e-14\n",
            "  Methyltransf_PK  53 krevkvaldcGaGiGrvtknlLlkllskvDlvepvekflekakeelaeekkkvgkvyvvglqeftPeegrYdliWiqWcvghLtDedlvaflkr 146\n",
            "                        e+k +l+  aGiGr t  +  + ++ v  ve +e+f++   ++ ++      k+ +  + +   e +++d+i+  W++  + De++ af k+\n",
            "   XP_002898137.1  87 CLENKDVLELAAGIGRYTSVI-ATKAKSVTAVEFIEDFIKVNADNNGH--LGNIKFLCKDVVNLEAEPNSFDVIFSNWILMYMEDEEVKAFAKK 177\n",
            "                      3467889**********9765.566899***********988866554..788899************************************** PP\n",
            "\n",
            "  Methyltransf_PK 147 ckkalkpnGlivvkenvake 166\n",
            "                      ++k l+p+G +  +e   k+\n",
            "   XP_002898137.1 178 AVKWLRPGGKLFFRESCFKQ 197\n",
            "                      *********99999877655 PP\n",
            "\n",
            "  == domain 2  score: 2.9 bits;  conditional E-value: 0.033\n",
            "  Methyltransf_PK  42 lkklkkeslpgkrevkvaldcGaGiGrvtknlLlkllskvDlvepvekflekakeelaeekkkvgkvyvvglqeftPeegrYdliWiqWcvghL 135\n",
            "                      ++kl+ +        +++ld+G+GiG     +  ++   v  ++   +++++a e+  ++ +   ++ ++   +   ++ ++d+++ +  + h+\n",
            "   XP_002898137.1 314 VEKLNLQ------PGERVLDVGCGIGGGDFYMARQFGVSVVGIDLSTNMVHRALETSMKDPSVDVEFEICDATKKEFADASFDVVYSRDTILHI 401\n",
            "                      4444444......446789999999987666666666566666666666666655444443333333333333334457789999999999999 PP\n",
            "\n",
            "  Methyltransf_PK 136 tDedlvaflkrckkalkpnGlivvke 161\n",
            "                       D++  a + +  + lkp+G + + +\n",
            "   XP_002898137.1 402 EDKE--ALFAKFFRWLKPGGRVLISD 425\n",
            "                      9988..45666777888888888776 PP\n",
            "\n",
            ">> XP_002898136.1  phosphoethanolamine N-methyltransferase [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   36.6   0.0   1.7e-12   3.3e-09      30     166 ..      26     162 ..       6     177 .. 0.81\n",
            "   2 ?    3.2   0.0     0.027        54      41     161 ..     280     392 ..     239     405 .. 0.69\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 36.6 bits;  conditional E-value: 1.7e-12\n",
            "  Methyltransf_PK  30 vskiDvkgsrkflkklkkeslp...gkrevkvaldcGaGiGrvtknlLlkllskvDlvepvekflekakeelaeekkkvgkvyvvglqeftPee 120\n",
            "                      ++++ + +  k l++l++ +        e+k +l+  aGiGr t  ++ k ++ +  ve +e+f ++     ++++     + +  + +   e \n",
            "   XP_002898136.1  26 IETMMLDSNAKVLHQLEMPEILeraPSMENKDVLELAAGIGRFTF-IIGKSAKSLTAVEFIEDFHKANV--ATNGHLDNVTFLCQDVVNLEAEP 116\n",
            "                      455556666677777766544322245678999**********95.7789999***********76544..344677888999*********** PP\n",
            "\n",
            "  Methyltransf_PK 121 grYdliWiqWcvghLtDedlvaflkrckkalkpnGlivvkenvake 166\n",
            "                      +++d+i+  W+   L De++ +f k+++k l+p+G +  +e   ++\n",
            "   XP_002898136.1 117 NSFDVIFSNWIFMYLGDEEVKNFAKKAVKWLRPGGKLFFRESCFRQ 162\n",
            "                      ***********************************99999876655 PP\n",
            "\n",
            "  == domain 2  score: 3.2 bits;  conditional E-value: 0.027\n",
            "  Methyltransf_PK  41 flkklkkeslpgkrevkvaldcGaGiGrvtknlLlkllskvDlvepvekflekakeelaeekkkvgkvyvvglqeftPeegrYdliWiqWcvgh 134\n",
            "                      f++kl+ +        +++ld+G+GiG     +  ++   v  ++   +++++a e+  ++ +   ++ ++   +   ++ ++d+++ +  + h\n",
            "   XP_002898136.1 280 FVEKLNLQ------PGERVLDVGCGIGGGDFYMARQFGVSVVGIDLSTNMVHRALETSMKDPSVDVEFEICDATKKEFADASFDVVYSRDTILH 367\n",
            "                      44444444......44678999999998766666666656666666666666665544444333334433433333445778999999999999 PP\n",
            "\n",
            "  Methyltransf_PK 135 LtDedlvaflkrckkalkpnGlivvke 161\n",
            "                      + D++  a + +  + lkp+G + + +\n",
            "   XP_002898136.1 368 IEDKE--ALFAKFFRWLKPGGRVLISD 392\n",
            "                      99988..45667777888888888776 PP\n",
            "\n",
            ">> XP_002908111.1  uncharacterized protein PITG_01446 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   18.5   0.0   5.6e-07    0.0011      55     194 ..      66     202 ..      57     221 .. 0.74\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 18.5 bits;  conditional E-value: 5.6e-07\n",
            "  Methyltransf_PK  55 evkvaldcGaGiGrvtknlLlkllskvDlvepvekflekakeelae..ekkkvgkvyvvglqeftPeegrYdliWiqWcvghLtDedlvaflkr 146\n",
            "                      + k +ld+G+G G  ++ l ++   kvD +e   ++ e+ + el++  + +   +     + e    +++Yd +    c  h+   +  + ++r\n",
            "   XP_002908111.1  66 NEKKVLDIGTGFGGTARLLAHRSGCKVDALELQPDLSEAGR-ELTRrcGLNDQVSHLSGDFLELPIRKKEYDAVIGLLCFLHIG--HWRQLFQR 156\n",
            "                      67889*********9999999999*******9888777666.8888453344444445555566667999****9999998885..77899*** PP\n",
            "\n",
            "  Methyltransf_PK 147 ckkalkpnGlivvkenvakeekdelDkeDssvtrseaklkklfkkagL 194\n",
            "                      c  +lkp+G++ v +   + e+  l +eD+   +++ +   l+++ ++\n",
            "   XP_002908111.1 157 CYDSLKPGGFLYVDDFFLRGEN--LTHEDKLTLKQDIYCSDLLRQEEI 202\n",
            "                      ***********99876666554..666666666666555555555444 PP\n",
            "\n",
            ">> XP_002905021.1  ribosomal protein S6 kinase, putative [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.6   0.0   3.6e-05     0.071      90     179 ..     253     344 ..     230     352 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.6 bits;  conditional E-value: 3.6e-05\n",
            "  Methyltransf_PK  90 flekakeelaeekkkvgkvyvvglqeftPeegrYdl..iWiqWcvghLtDedlvaflkrckkalkpnGlivvkenvakeekdelDkeDssvt 179\n",
            "                      ++e++++++ e +++++ v  +   ++   +++Y l  +  qWc++ L+  + ++ lkr k+   p G++v+    a++++d l    ssv \n",
            "   XP_002905021.1 253 LVEALRSNMKEMQRHISGVVFLQSTHHISSDDSYTLrkMIAQWCLAYLSSPEGLKALKRMKSREMPLGCVVLSAGRANSDVDLLQTVVSSVY 344\n",
            "                      56777777777666666666666666677777887755678******************************************999998885 PP\n",
            "\n",
            ">> XP_002905459.1  uncharacterized protein PITG_04729 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.2   0.0   2.4e-05     0.047      55     163 ..     170     282 ..     153     293 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.2 bits;  conditional E-value: 2.4e-05\n",
            "  Methyltransf_PK  55 evkvaldcGaGiGr...vtknlLlkllskvDlvepvekflekakeelaeekkkvgkvyvvglqeftPeegrYdliWiqWcvghLt.Dedlvafl 144\n",
            "                      + k +ld G+G G    v+k++  + l k  +vep ++++++a+  l        +  ++ l     + ++Ydl+ + ++ + +t D + va  \n",
            "   XP_002905459.1 170 KPKSVLDFGSGPGTaswVAKDFYDQSLDKYRVVEPSQSMVDAAEVVLEGFPGLSMRRNITDLSRDIDAGNKYDLVVVSYVFSDITnDFERVATT 263\n",
            "                      457899*****99644489*************************8887733334444556677777779**************99355899*99 PP\n",
            "\n",
            "  Methyltransf_PK 145 krckkalkpnGlivvkenv 163\n",
            "                      +   + l ++G++v+ +  \n",
            "   XP_002905459.1 264 SALWELLSEDGCLVIVDRG 282\n",
            "                      **************98865 PP\n",
            "\n",
            ">> XP_002905473.1  uncharacterized protein PITG_04747 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.2   0.0   2.4e-05     0.047      55     163 ..     176     288 ..     159     301 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.2 bits;  conditional E-value: 2.4e-05\n",
            "  Methyltransf_PK  55 evkvaldcGaGiGr...vtknlLlkllskvDlvepvekflekakeelaeekkkvgkvyvvglqeftPeegrYdliWiqWcvghLt.Dedlvafl 144\n",
            "                      + k +ld G+G G    v+k++  + l k  +vep ++++++a+  l        +  ++ l     + ++Ydl+ + ++ + +t D + va  \n",
            "   XP_002905473.1 176 KPKSVLDFGSGPGTaswVAKDFYDQSLDKYRVVEPSQSMVDAAEVVLEGFPGLSMRRNITDLSRDIDAGNKYDLVVVSYVFSDITnDFERVATT 269\n",
            "                      457899*****99644489*************************8887733334444556677777779**************99355899*9* PP\n",
            "\n",
            "  Methyltransf_PK 145 krckkalkpnGlivvkenv 163\n",
            "                      +   + l ++G++v+ +  \n",
            "   XP_002905473.1 270 SALWELLSEDGCLVIVDRG 288\n",
            "                      **************98865 PP\n",
            "\n",
            ">> XP_002900712.1  uncharacterized protein PITG_12077 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.2   0.0   2.4e-05     0.047      55     163 ..     176     288 ..     159     301 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.2 bits;  conditional E-value: 2.4e-05\n",
            "  Methyltransf_PK  55 evkvaldcGaGiGr...vtknlLlkllskvDlvepvekflekakeelaeekkkvgkvyvvglqeftPeegrYdliWiqWcvghLt.Dedlvafl 144\n",
            "                      + k +ld G+G G    v+k++  + l k  +vep ++++++a+  l        +  ++ l     + ++Ydl+ + ++ + +t D + va  \n",
            "   XP_002900712.1 176 KPKSVLDFGSGPGTaswVAKDFYDQSLDKYRVVEPSQSMVDAAEVVLEGFPGLSMRRNITDLSRDIDAGNKYDLVVVSYVFSDITnDFERVATT 269\n",
            "                      457899*****99644489*************************8887733334444556677777779**************99355899*9* PP\n",
            "\n",
            "  Methyltransf_PK 145 krckkalkpnGlivvkenv 163\n",
            "                      +   + l ++G++v+ +  \n",
            "   XP_002900712.1 270 SALWELLSEDGCLVIVDRG 288\n",
            "                      **************98865 PP\n",
            "\n",
            ">> XP_002905524.1  uncharacterized protein PITG_04818 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.8   0.0    0.0043       8.5      55     131 ..      31     109 ..       4     131 .. 0.78\n",
            "   2 ?    4.4   0.0     0.011        23     138     166 ..     243     271 ..     232     276 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.8 bits;  conditional E-value: 0.0043\n",
            "  Methyltransf_PK  55 evkvaldcGaGiGrvtknlLl..kllskvDlvepvekflekakeelaeekkkvgkvyvvglqeftPeegrYdliW.iqWc 131\n",
            "                      ++++  ++G   G++  nlL+  + ++ v+  e+v ++  k++  l  +   v+++ v+ +++ tP++g+++li  +q +\n",
            "   XP_002905524.1  31 NQNITSNTGQYTGQIQFNLLTisSQAAFVNWEEAVIELPVKLQ-ILNGGAGSVTSTAVCTIDQLTPKAGEWQLIDsVQAV 109\n",
            "                      4455667788888999998872244578999999999999998.7777899*********************99424555 PP\n",
            "\n",
            "  == domain 2  score: 4.4 bits;  conditional E-value: 0.011\n",
            "  Methyltransf_PK 138 edlvaflkrckkalkpnGlivvkenvake 166\n",
            "                      +d+ +++k+c  + +p G+i +  n +++\n",
            "   XP_002905524.1 243 QDICDYFKKCPMQKNPRGFIYLNYNSSST 271\n",
            "                      7999******************9998765 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (218 nodes)\n",
            "Target sequences:                      17797  (7477560 residues searched)\n",
            "Passed MSV filter:                       442  (0.0248356); expected 355.9 (0.02)\n",
            "Passed bias filter:                      368  (0.0206776); expected 355.9 (0.02)\n",
            "Passed Vit filter:                        37  (0.002079); expected 17.8 (0.001)\n",
            "Passed Fwd filter:                         9  (0.000505703); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              17797  [actual number of targets]\n",
            "Domain search space  (domZ):               9  [number of targets reported over threshold]\n",
            "# CPU time: 0.24u 0.01s 00:00:00.25 Elapsed: 00:00:00.27\n",
            "# Mc/sec: 5983.42\n",
            "//\n",
            "Query:       YTH  [M=183]\n",
            "Accession:   PF04146.18\n",
            "Description: YT521-B-like domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    2.7e-28   99.4   0.6      9e-19   68.3   0.2    2.1  2  XP_002908404.1  uncharacterized protein PITG_01795 [Phytophth\n",
            "    6.7e-23   81.8   0.0    1.6e-21   77.3   0.0    2.2  1  XP_002902266.1  uncharacterized protein PITG_10481 [Phytophth\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_002908404.1  uncharacterized protein PITG_01795 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   68.3   0.2     1e-22     9e-19       2      66 ..      32     100 ..      31     113 .. 0.90\n",
            "   2 !   29.7   0.0   7.3e-11   6.5e-07     128     182 ..     112     167 ..     108     168 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 68.3 bits;  conditional E-value: 1e-22\n",
            "                     EEEEEEESHHHHHHHHHHHCEEEEEHHHHHHHHHHHHCS....-EEEEEEEETTTSEEEEEEEEE.... CS\n",
            "             YTH   2 ryFilKslteedlekSvetgiWatqkhnekvLdqafrts....ekVyLiFsvnksgeFfGyArmtssss 66 \n",
            "                     r+F+lKs++e++ +kS+++giW+t++ ++  Ldq f++     + V ++Fsv ++++F G+Armts  +\n",
            "  XP_002908404.1  32 RCFVLKSFSEANFHKSLKYGIWSTTTMHNALLDQVFKSDltavRPVLFFFSVCGTKHFNGVARMTSGVR 100\n",
            "                     99************************************9999***********************6332 PP\n",
            "\n",
            "  == domain 2  score: 29.7 bits;  conditional E-value: 7.3e-11\n",
            "                     .--EEEEEEEEEEEEEEGHHCCCHCHCC.CTT-BGCG--TTEEE-HHHHHHHHHHH CS\n",
            "             YTH 128 swgkpFkveWlstkrlpfqrtrhlrnpw.nenreVkvsrDgtelepsvGrqLlelf 182\n",
            "                     ++++ F+veWl +k++p++ + ++r ++  ++++++ +rD++e+  + +++ l++f\n",
            "  XP_002908404.1 112 KYEGFFQVEWLLVKDVPNYVFTGVRMSNtPSKKSITSCRDCEEVLFEEANEFLSIF 167\n",
            "                     57789*********************995589********************9998 PP\n",
            "\n",
            ">> XP_002902266.1  uncharacterized protein PITG_10481 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   77.3   0.0   1.8e-25   1.6e-21       1     181 [.     251     391 ..     251     393 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 77.3 bits;  conditional E-value: 1.8e-25\n",
            "                     EEEEEEEESHHHHHHHHHHHCEEEEEHHHHHHHHHHHHCS-EEEEEEEETTTSEEEEEEEEE......SEEBTTS-................... CS\n",
            "             YTH   1 kryFilKslteedlekSvetgiWatqkhnekvLdqafrtsekVyLiFsvnksgeFfGyArmtsssssssavefassaseesaseessaaaeesar 95 \n",
            "                     ++yF+l+s +  +l++Sv+++ Wa+++   ++++ a +ts++V+L+F v  s++++G+Ar++     + a++ a+                   +\n",
            "  XP_002902266.1 251 AKYFLLQSASYLNLAHSVHHERWAVPPAVLQQIKMASETSDEVFLFFAVGPSKHYQGVARLV-----NGAMSSADA------------------S 322\n",
            "                     59************************************************************.....222222222..................3 PP\n",
            "\n",
            "                     ........GGCC-.TTEE...............--EEEEEEEEEEEEEEGHHCCCHCHCCCTT-BGCG--TTEEE-HHHHHHHHHH CS\n",
            "             YTH  96 sidasaegssfweaesskeeasskaeeeeesdswgkpFkveWlstkrlpfqrtrhlrnpwnenreVkvsrDgtelepsvGrqLlel 181\n",
            "                     + ++ a+g + +ea+ +              ++w ++F +eWl++++ p +r+ +++n+      V+ + +g+el+ ++G++L++l\n",
            "  XP_002902266.1 323 AGEDLAAGVVPYEADGK--------------AEWAGSFGIEWLRICECPWERLAQFENKQ---LAVSECPNGQELDAETGHALVRL 391\n",
            "                     34556777667775554..............58**********************99985...89******************987 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (183 nodes)\n",
            "Target sequences:                      17797  (7477560 residues searched)\n",
            "Passed MSV filter:                      1776  (0.0997921); expected 355.9 (0.02)\n",
            "Passed bias filter:                      874  (0.0491094); expected 355.9 (0.02)\n",
            "Passed Vit filter:                        98  (0.00550655); expected 17.8 (0.001)\n",
            "Passed Fwd filter:                         3  (0.000168568); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              17797  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.36u 0.01s 00:00:00.37 Elapsed: 00:00:00.34\n",
            "# Mc/sec: 3934.25\n",
            "//\n",
            "Query:       zf-CCCH  [M=27]\n",
            "Accession:   PF00642.27\n",
            "Description: Zinc finger C-x8-C-x5-C-x3-H type (and similar)\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "      5e-31  106.0  36.5    8.3e-07   28.8   2.6    5.5  5  XP_002902272.1  uncharacterized protein PITG_10487 [Phytophth\n",
            "    9.6e-21   73.2  45.1    8.9e-05   22.3   1.4    5.3  5  XP_002905403.1  uncharacterized protein PITG_04664 [Phytophth\n",
            "    5.9e-19   67.5  17.2    3.1e-11   42.9   5.3    2.5  2  XP_002906208.1  uncharacterized protein PITG_03115 [Phytophth\n",
            "    2.5e-17   62.3  22.0    4.5e-10   39.2   2.1    4.0  4  XP_002902266.1  uncharacterized protein PITG_10481 [Phytophth\n",
            "    5.4e-13   48.5  17.2      2e-07   30.8   2.3    3.1  3  XP_002999307.1  splicing factor U2AF 35 kDa subunit [Phytopht\n",
            "    2.2e-12   46.5  19.8    8.3e-06   25.6   5.5    3.5  3  XP_002895137.1  uncharacterized protein PITG_21380 [Phytophth\n",
            "    1.9e-11   43.5  26.2    7.2e-05   22.6   2.6    3.9  3  XP_002900845.1  makorin-like protein [Phytophthora infestans \n",
            "    4.7e-11   42.3  16.3    3.3e-05   23.7   5.2    3.2  2  XP_002906446.1  uncharacterized protein PITG_03374 [Phytophth\n",
            "    2.1e-10   40.2  13.4    1.3e-05   25.0   1.6    2.6  2  XP_002998703.1  uncharacterized protein PITG_06616 [Phytophth\n",
            "    7.6e-10   38.5   9.3    9.5e-06   25.4   1.0    2.4  2  XP_002898513.1  uncharacterized protein PITG_14610 [Phytophth\n",
            "    2.7e-09   36.7   2.7    2.7e-09   36.7   2.7    2.0  2  XP_002905607.1  uncharacterized protein PITG_04905 [Phytophth\n",
            "    4.1e-07   29.7   2.7      9e-07   28.7   2.7    1.6  1  XP_002908299.1  uncharacterized protein PITG_01678 [Phytophth\n",
            "    4.5e-07   29.6   9.8     0.0012   18.7   1.0    2.5  2  XP_002899948.1  RNA-binding protein NOB1 [Phytophthora infest\n",
            "    5.7e-07   29.3  10.6    3.6e-05   23.5   3.1    2.5  2  XP_002897890.1  uncharacterized protein PITG_16118 [Phytophth\n",
            "    6.5e-07   29.1  23.4     0.0049   16.8   5.3    4.7  3  XP_002905007.1  uncharacterized protein PITG_07041 [Phytophth\n",
            "    7.3e-06   25.8   0.6    7.3e-06   25.8   0.6    2.2  2  XP_002905360.1  uncharacterized protein PITG_04608 [Phytophth\n",
            "    1.2e-05   25.1  12.4     0.0069   16.3   2.5    2.5  2  XP_002907667.1  uncharacterized protein PITG_02775 [Phytophth\n",
            "    1.3e-05   25.0   9.6     0.0022   17.9   0.9    2.6  2  XP_002907473.1  tRNA-dihydrouridine synthase 3 [Phytophthora \n",
            "    1.3e-05   24.9  12.4     0.0071   16.2   2.5    2.5  2  XP_002896200.1  uncharacterized protein PITG_19639 [Phytophth\n",
            "    4.3e-05   23.3   4.2    8.6e-05   22.3   4.2    1.6  1  XP_002997690.1  uncharacterized protein PITG_18056 [Phytophth\n",
            "      5e-05   23.1  28.1    0.00017   21.4   4.7    3.4  2  XP_002902613.1  uncharacterized protein PITG_10082 [Phytophth\n",
            "    0.00035   20.4   6.8      0.095   12.6   1.2    2.5  2  XP_002908060.1  uncharacterized protein PITG_01390 [Phytophth\n",
            "     0.0005   19.9   1.0     0.0005   19.9   1.0    3.1  3  XP_002906927.1  uncharacterized protein PITG_03888 [Phytophth\n",
            "     0.0017   18.2   1.9     0.0039   17.1   1.9    1.6  1  XP_002907409.1  uncharacterized protein PITG_02485 [Phytophth\n",
            "     0.0051   16.7   7.2      0.011   15.7   7.2    1.6  1  XP_002897011.1  uncharacterized protein PITG_17110 [Phytophth\n",
            "     0.0082   16.0   2.6     0.0082   16.0   2.6    2.4  2  XP_002904939.1  uncharacterized protein PITG_06966 [Phytophth\n",
            "  ------ inclusion threshold ------\n",
            "      0.022   14.7   4.3      0.057   13.3   4.3    1.8  1  XP_002909043.1  uncharacterized protein PITG_00444 [Phytophth\n",
            "      0.041   13.8   4.8       0.13   12.2   1.1    3.2  2  XP_002908931.1  uncharacterized protein PITG_00313 [Phytophth\n",
            "       0.39   10.7   1.7          2    8.4   1.4    2.3  2  XP_002906972.1  uncharacterized protein PITG_03939 [Phytophth\n",
            "        8.7    6.4  12.8          7    6.7   1.2    2.7  2  XP_002905649.1  uncharacterized protein PITG_04956 [Phytophth\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_002902272.1  uncharacterized protein PITG_10487 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   28.8   2.6   1.4e-09   8.3e-07       5      26 ..     215     235 ..     212     236 .. 0.91\n",
            "   2 !   24.9   1.4   2.2e-08   1.3e-05       4      25 ..     284     304 ..     281     306 .. 0.92\n",
            "   3 !   23.6   2.0     6e-08   3.6e-05       4      25 ..     342     362 ..     339     363 .. 0.91\n",
            "   4 !   27.8   1.0   2.9e-09   1.7e-06       4      25 ..     395     415 ..     393     416 .. 0.94\n",
            "   5 !   22.4   0.9   1.4e-07   8.5e-05       4      25 ..     440     461 ..     438     462 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 28.8 bits;  conditional E-value: 1.4e-09\n",
            "                     CBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH   5 lCkffmrtGyCkyGdrCkFaHg 26 \n",
            "                      C+ f++ G+C +Gd+CkFaH+\n",
            "  XP_002902272.1 215 VCHKFQQ-GQCTRGDACKFAHV 235\n",
            "                     7******.*************7 PP\n",
            "\n",
            "  == domain 2  score: 24.9 bits;  conditional E-value: 2.2e-08\n",
            "                     TCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH   4 elCkffmrtGyCkyGdrCkFaH 25 \n",
            "                     + C +f++ G Ck+G++CkF H\n",
            "  XP_002902272.1 284 PVCINFQK-GKCKRGAACKFQH 304\n",
            "                     78******.***********88 PP\n",
            "\n",
            "  == domain 3  score: 23.6 bits;  conditional E-value: 6e-08\n",
            "                     TCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH   4 elCkffmrtGyCkyGdrCkFaH 25 \n",
            "                     + C ++++ G Ck+G++C+F H\n",
            "  XP_002902272.1 342 PVCENYQK-GKCKRGAACRFRH 362\n",
            "                     79******.***********88 PP\n",
            "\n",
            "  == domain 4  score: 27.8 bits;  conditional E-value: 2.9e-09\n",
            "                     TCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH   4 elCkffmrtGyCkyGdrCkFaH 25 \n",
            "                     + C++f+r G+C +G++C+FaH\n",
            "  XP_002902272.1 395 AVCRNFQR-GSCMRGASCRFAH 415\n",
            "                     78******.************* PP\n",
            "\n",
            "  == domain 5  score: 22.4 bits;  conditional E-value: 1.4e-07\n",
            "                     TCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH   4 elCkffmrtGyCkyGdrCkFaH 25 \n",
            "                     + C +++ tG+C +G++C+F H\n",
            "  XP_002902272.1 440 SVCYNWQSTGSCARGENCPFQH 461\n",
            "                     78******************98 PP\n",
            "\n",
            ">> XP_002905403.1  uncharacterized protein PITG_04664 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   19.5   3.4   1.1e-06   0.00067       2      25 ..      15      37 ..      14      38 .. 0.92\n",
            "   2 !   18.5   2.0   2.2e-06    0.0013       5      26 ..      49      69 ..      47      70 .. 0.96\n",
            "   3 !   21.6   2.2   2.6e-07   0.00015       5      26 ..      96     116 ..      94     117 .. 0.92\n",
            "   4 !   22.3   1.4   1.5e-07   8.9e-05       4      26 ..     214     235 ..     213     236 .. 0.94\n",
            "   5 !   14.7   5.7   3.6e-05     0.021       5      25 ..     345     366 ..     344     368 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 19.5 bits;  conditional E-value: 1.1e-06\n",
            "                    SSTCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH  2 ktelCkffmrtGyCkyGdrCkFaH 25\n",
            "                    k+ +C ff   G C+ G++CkF H\n",
            "  XP_002905403.1 15 KMTPCSFFVA-GKCRNGSSCKFFH 37\n",
            "                    789***8888.***********88 PP\n",
            "\n",
            "  == domain 2  score: 18.5 bits;  conditional E-value: 2.2e-06\n",
            "                    CBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH  5 lCkffmrtGyCkyGdrCkFaHg 26\n",
            "                    +Ckff+r G+C  G  CkF+H+\n",
            "  XP_002905403.1 49 PCKFFLR-GTCTAGRGCKFSHS 69\n",
            "                    8******.*************7 PP\n",
            "\n",
            "  == domain 3  score: 21.6 bits;  conditional E-value: 2.6e-07\n",
            "                     CBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH   5 lCkffmrtGyCkyGdrCkFaHg 26 \n",
            "                     +Ck f+++G C  Gd+C++ H \n",
            "  XP_002905403.1  96 PCK-FFKYGDCANGDKCPYLHN 116\n",
            "                     899.9999*************7 PP\n",
            "\n",
            "  == domain 4  score: 22.3 bits;  conditional E-value: 1.5e-07\n",
            "                     TCBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH   4 elCkffmrtGyCkyGdrCkFaHg 26 \n",
            "                     ++C ff++ G C+yG++C +aH+\n",
            "  XP_002905403.1 214 KICTFFLQ-GLCRYGNTCFYAHS 235\n",
            "                     58******.*************8 PP\n",
            "\n",
            "  == domain 5  score: 14.7 bits;  conditional E-value: 3.6e-05\n",
            "                     CBSHHHHTS.--TTGGG-SSBS CS\n",
            "         zf-CCCH   5 lCkffmrtG.yCkyGdrCkFaH 25 \n",
            "                     +C++f +++ +C++G +C +aH\n",
            "  XP_002905403.1 345 PCRHFDEGRgTCPFGTSCFYAH 366\n",
            "                     7******88************* PP\n",
            "\n",
            ">> XP_002906208.1  uncharacterized protein PITG_03115 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   42.9   5.3   5.2e-14   3.1e-11       1      27 []      43      69 ..      43      69 .. 0.97\n",
            "   2 !   30.1   4.0   5.2e-10   3.1e-07       1      25 [.      81     105 ..      81     107 .. 0.96\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 42.9 bits;  conditional E-value: 5.2e-14\n",
            "                    TSSTCBSHHHHTS--TTGGG-SSBSCT CS\n",
            "         zf-CCCH  1 yktelCkffmrtGyCkyGdrCkFaHgq 27\n",
            "                    yktelCk+f ++G+C+yG++C+FaHg+\n",
            "  XP_002906208.1 43 YKTELCKHFTENGSCRYGSKCQFAHGE 69\n",
            "                    9************************96 PP\n",
            "\n",
            "  == domain 2  score: 30.1 bits;  conditional E-value: 5.2e-10\n",
            "                     TSSTCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH   1 yktelCkffmrtGyCkyGdrCkFaH 25 \n",
            "                     ykt +Ck f+ tG C yG+rC+F+H\n",
            "  XP_002906208.1  81 YKTTRCKAFLSTGKCMYGSRCRFIH 105\n",
            "                     9************************ PP\n",
            "\n",
            ">> XP_002902266.1  uncharacterized protein PITG_10481 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.3   0.2   2.3e-05     0.014       2      25 ..      61      83 ..      60      84 .. 0.96\n",
            "   2 ?   -2.5   0.1       9.1   5.4e+03      20      26 ..     103     109 ..     101     110 .. 0.73\n",
            "   3 !   18.5   4.2   2.3e-06    0.0014       3      25 ..     114     135 ..     112     137 .. 0.89\n",
            "   4 !   39.2   2.1   7.6e-13   4.5e-10       1      27 []     173     199 ..     173     199 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.3 bits;  conditional E-value: 2.3e-05\n",
            "                    SSTCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH  2 ktelCkffmrtGyCkyGdrCkFaH 25\n",
            "                    +t+ C++++r   C  Gd+C+F H\n",
            "  XP_002902266.1 61 GTVVCRHWLR-ALCMKGDNCEFLH 83\n",
            "                    699*******.************* PP\n",
            "\n",
            "  == domain 2  score: -2.5 bits;  conditional E-value: 9.1\n",
            "                     G-SSBSC CS\n",
            "         zf-CCCH  20 rCkFaHg 26 \n",
            "                      C+F H+\n",
            "  XP_002902266.1 103 ECPFRHV 109\n",
            "                     6888886 PP\n",
            "\n",
            "  == domain 3  score: 18.5 bits;  conditional E-value: 2.3e-06\n",
            "                     STCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH   3 telCkffmrtGyCkyGdrCkFaH 25 \n",
            "                     +++C f+++ G+C++G++C++ H\n",
            "  XP_002902266.1 114 RVECAFYKQ-GFCSHGSSCRYRH 135\n",
            "                     789**9999.**********988 PP\n",
            "\n",
            "  == domain 4  score: 39.2 bits;  conditional E-value: 7.6e-13\n",
            "                     TSSTCBSHHHHTS--TTGGG-SSBSCT CS\n",
            "         zf-CCCH   1 yktelCkffmrtGyCkyGdrCkFaHgq 27 \n",
            "                     +k ++Ck++ ++G+C++Gd C+FaHg+\n",
            "  XP_002902266.1 173 FKIAICKHWEKMGSCPFGDECHFAHGE 199\n",
            "                     5889*********************96 PP\n",
            "\n",
            ">> XP_002999307.1  splicing factor U2AF 35 kDa subunit [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   30.8   2.3   3.3e-10     2e-07       2      26 ..      69      93 ..      68      94 .. 0.93\n",
            "   2 ?   -1.5   0.1       4.3   2.6e+03       4       8 ..     143     147 ..     141     148 .. 0.55\n",
            "   3 !   25.0   4.0   2.1e-08   1.2e-05       2      25 ..     205     227 ..     204     229 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 30.8 bits;  conditional E-value: 3.3e-10\n",
            "                    SSTCBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH  2 ktelCkffmrtGyCkyGdrCkFaHg 26\n",
            "                    ++++C+f+++ G+C++GdrC++ H \n",
            "  XP_002999307.1 69 DRVNCPFYFKIGACRHGDRCSRLHN 93\n",
            "                    689*********************7 PP\n",
            "\n",
            "  == domain 2  score: -1.5 bits;  conditional E-value: 4.3\n",
            "                     TCBSH CS\n",
            "         zf-CCCH   4 elCkf 8  \n",
            "                     elCkf\n",
            "  XP_002999307.1 143 ELCKF 147\n",
            "                     55554 PP\n",
            "\n",
            "  == domain 3  score: 25.0 bits;  conditional E-value: 2.1e-08\n",
            "                     SSTCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH   2 ktelCkffmrtGyCkyGdrCkFaH 25 \n",
            "                     ++++C+ f + G+C++G+ C+F+H\n",
            "  XP_002999307.1 205 REARCRQFDE-GTCNRGGYCNFMH 227\n",
            "                     6899******.************* PP\n",
            "\n",
            ">> XP_002895137.1  uncharacterized protein PITG_21380 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   23.3   2.4   7.2e-08   4.3e-05       1      19 [.      63      81 ..      63      82 .. 0.96\n",
            "   2 ?    5.4   0.1      0.03        18      19      26 ..     105     112 ..     105     113 .. 0.88\n",
            "   3 !   25.6   5.5   1.4e-08   8.3e-06       1      25 [.     125     149 ..     125     151 .. 0.96\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 23.3 bits;  conditional E-value: 7.2e-08\n",
            "                    TSSTCBSHHHHTS--TTGG CS\n",
            "         zf-CCCH  1 yktelCkffmrtGyCkyGd 19\n",
            "                    yktelCk+f++ G C+yG+\n",
            "  XP_002895137.1 63 YKTELCKRFSEFGNCRYGG 81\n",
            "                    9*****************7 PP\n",
            "\n",
            "  == domain 2  score: 5.4 bits;  conditional E-value: 0.03\n",
            "                     GG-SSBSC CS\n",
            "         zf-CCCH  19 drCkFaHg 26 \n",
            "                     ++C+FaHg\n",
            "  XP_002895137.1 105 AKCQFAHG 112\n",
            "                     58*****8 PP\n",
            "\n",
            "  == domain 3  score: 25.6 bits;  conditional E-value: 1.4e-08\n",
            "                     TSSTCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH   1 yktelCkffmrtGyCkyGdrCkFaH 25 \n",
            "                     ykt  Ck +  +G+C+yG+rC+F+H\n",
            "  XP_002895137.1 125 YKTTKCKSYWGSGHCPYGSRCRFIH 149\n",
            "                     9************************ PP\n",
            "\n",
            ">> XP_002900845.1  makorin-like protein [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   22.6   2.6   1.2e-07   7.2e-05       2      25 ..      10      32 ..       9      34 .. 0.93\n",
            "   2 !   21.4   1.3   2.9e-07   0.00017       5      22 ..      57      73 ..      55      74 .. 0.94\n",
            "   3 !    9.9   7.3    0.0012      0.71       5      25 ..     176     197 ..     174     199 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 22.6 bits;  conditional E-value: 1.2e-07\n",
            "                    SSTCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH  2 ktelCkffmrtGyCkyGdrCkFaH 25\n",
            "                     t+lC+f+   G C+yG+ C F+H\n",
            "  XP_002900845.1 10 TTALCRFYVL-GKCRYGSCCTFSH 32\n",
            "                    599***9999.************* PP\n",
            "\n",
            "  == domain 2  score: 21.4 bits;  conditional E-value: 2.9e-07\n",
            "                    CBSHHHHTS--TTGGG-S CS\n",
            "         zf-CCCH  5 lCkffmrtGyCkyGdrCk 22\n",
            "                     C+f++r G CkyGd C+\n",
            "  XP_002900845.1 57 DCPFYLR-GNCKYGDHCR 73\n",
            "                    7******.*********7 PP\n",
            "\n",
            "  == domain 3  score: 9.9 bits;  conditional E-value: 0.0012\n",
            "                     CBSHHH.HTS--TTGGG-SSBS CS\n",
            "         zf-CCCH   5 lCkffm.rtGyCkyGdrCkFaH 25 \n",
            "                     +Ck+f+   G+C++G+rC +aH\n",
            "  XP_002900845.1 176 ECKYFNgLFGSCPFGPRCFYAH 197\n",
            "                     6777765689************ PP\n",
            "\n",
            ">> XP_002906446.1  uncharacterized protein PITG_03374 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   23.7   5.2   5.6e-08   3.3e-05       2      25 ..     171     193 ..     170     195 .. 0.92\n",
            "   2 !   23.1   3.2   8.5e-08   5.1e-05       2      25 ..     237     259 ..     236     261 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 23.7 bits;  conditional E-value: 5.6e-08\n",
            "                     SSTCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH   2 ktelCkffmrtGyCkyGdrCkFaH 25 \n",
            "                     ++++C+ff r G C+ G++CkF+H\n",
            "  XP_002906446.1 171 GQRPCRFFVR-GNCRDGEKCKFSH 193\n",
            "                     579*******.************* PP\n",
            "\n",
            "  == domain 2  score: 23.1 bits;  conditional E-value: 8.5e-08\n",
            "                     SSTCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH   2 ktelCkffmrtGyCkyGdrCkFaH 25 \n",
            "                     +t+ Ckff++   C+ Gd+CkF+H\n",
            "  XP_002906446.1 237 RTRKCKFFAQ-SKCRDGDKCKFSH 259\n",
            "                     5899******.************* PP\n",
            "\n",
            ">> XP_002998703.1  uncharacterized protein PITG_06616 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   23.7   0.7   5.5e-08   3.2e-05       4      25 ..     206     226 ..     204     228 .. 0.91\n",
            "   2 !   25.0   1.6   2.1e-08   1.3e-05       3      26 ..     243     266 ..     242     267 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 23.7 bits;  conditional E-value: 5.5e-08\n",
            "                     TCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH   4 elCkffmrtGyCkyGdrCkFaH 25 \n",
            "                     + C +fmr G+Ck GdrC+F H\n",
            "  XP_002998703.1 206 KVCYDFMR-GECKWGDRCNFEH 226\n",
            "                     68******.***********88 PP\n",
            "\n",
            "  == domain 2  score: 25.0 bits;  conditional E-value: 2.1e-08\n",
            "                     STCBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH   3 telCkffmrtGyCkyGdrCkFaHg 26 \n",
            "                     ++ C++f rt  C++Gd+C ++H+\n",
            "  XP_002998703.1 243 RRVCDNFIRTKNCRFGDKCLYSHD 266\n",
            "                     799********************7 PP\n",
            "\n",
            ">> XP_002898513.1  uncharacterized protein PITG_14610 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.3   1.8   5.4e-06    0.0032       2      26 ..      83     106 ..      82     107 .. 0.95\n",
            "   2 !   25.4   1.0   1.6e-08   9.5e-06       2      26 ..     118     141 ..     117     142 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.3 bits;  conditional E-value: 5.4e-06\n",
            "                     SSTCBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH   2 ktelCkffmrtGyCkyGdrCkFaHg 26 \n",
            "                      te+C+ f+  G+C+ Gd Ck+ H+\n",
            "  XP_002898513.1  83 TTEPCRGFAA-GICRKGDMCKYYHP 106\n",
            "                     59********.*************8 PP\n",
            "\n",
            "  == domain 2  score: 25.4 bits;  conditional E-value: 1.6e-08\n",
            "                     SSTCBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH   2 ktelCkffmrtGyCkyGdrCkFaHg 26 \n",
            "                      t lC++f r G+C  G++C FaH+\n",
            "  XP_002898513.1 118 STQLCRHFTR-GFCAQGSACTFAHV 141\n",
            "                     689*******.*************7 PP\n",
            "\n",
            ">> XP_002905607.1  uncharacterized protein PITG_04905 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   36.7   2.7   4.5e-12   2.7e-09       1      26 [.     161     186 ..     161     187 .. 0.92\n",
            "   2 ?   -2.3   0.1       7.5   4.5e+03       7      15 ..     265     273 ..     265     273 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 36.7 bits;  conditional E-value: 4.5e-12\n",
            "                     TSSTCBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH   1 yktelCkffmrtGyCkyGdrCkFaHg 26 \n",
            "                     y+   Ck++++tG+C yGd+CkF H+\n",
            "  XP_002905607.1 161 YQPDVCKDYKETGFCGYGDSCKFLHD 186\n",
            "                     67789********************7 PP\n",
            "\n",
            "  == domain 2  score: -2.3 bits;  conditional E-value: 7.5\n",
            "                     SHHHHTS-- CS\n",
            "         zf-CCCH   7 kffmrtGyC 15 \n",
            "                     k+f++t  C\n",
            "  XP_002905607.1 265 KNFKKTSRC 273\n",
            "                     578888777 PP\n",
            "\n",
            ">> XP_002908299.1  uncharacterized protein PITG_01678 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   28.7   2.7   1.5e-09     9e-07       4      26 ..       9      30 ..       7      31 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 28.7 bits;  conditional E-value: 1.5e-09\n",
            "                    TCBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH  4 elCkffmrtGyCkyGdrCkFaHg 26\n",
            "                    ++Ckff++ G+C+ G++C+F+H+\n",
            "  XP_002908299.1  9 KPCKFFLQ-GTCRNGNNCRFSHD 30\n",
            "                    79******.*************6 PP\n",
            "\n",
            ">> XP_002899948.1  RNA-binding protein NOB1 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.1   2.2   2.7e-05     0.016       5      26 ..     178     199 ..     177     200 .. 0.86\n",
            "   2 !   18.7   1.0     2e-06    0.0012       5      25 ..     220     240 ..     219     242 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.1 bits;  conditional E-value: 2.7e-05\n",
            "                     CBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH   5 lCkffmrtGyCkyGdrCkFaHg 26 \n",
            "                     +Ck+f     CkyGd C+F H+\n",
            "  XP_002899948.1 178 PCKYFGTAMGCKYGDECRFTHD 199\n",
            "                     6888888779*********997 PP\n",
            "\n",
            "  == domain 2  score: 18.7 bits;  conditional E-value: 2e-06\n",
            "                     CBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH   5 lCkffmrtGyCkyGdrCkFaH 25 \n",
            "                     +C+ff+    CkyGd C+F H\n",
            "  XP_002899948.1 220 PCRFFNTPEGCKYGDDCSFVH 240\n",
            "                     799999999************ PP\n",
            "\n",
            ">> XP_002897890.1  uncharacterized protein PITG_16118 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !    9.6   0.9    0.0014      0.83       3      19 ..     200     216 ..     199     216 .. 0.90\n",
            "   2 !   23.5   3.1   6.1e-08   3.6e-05       4      25 ..     224     245 ..     222     245 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.6 bits;  conditional E-value: 0.0014\n",
            "                     STCBSHHHHTS--TTGG CS\n",
            "         zf-CCCH   3 telCkffmrtGyCkyGd 19 \n",
            "                      + C++fm+   C +G+\n",
            "  XP_002897890.1 200 KAVCRTFMQKKKCTRGG 216\n",
            "                     579************96 PP\n",
            "\n",
            "  == domain 2  score: 23.5 bits;  conditional E-value: 6.1e-08\n",
            "                     TCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH   4 elCkffmrtGyCkyGdrCkFaH 25 \n",
            "                      lC +f++tG+Ck+Gd C+ +H\n",
            "  XP_002897890.1 224 QLCLYFFKTGQCKFGDDCRKNH 245\n",
            "                     69****************9555 PP\n",
            "\n",
            ">> XP_002905007.1  uncharacterized protein PITG_07041 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   12.9   3.9   0.00013      0.08       1      27 []     455     481 ..     455     481 .. 0.81\n",
            "   2 ?    6.2   0.4     0.017        10       2      16 ..     505     518 ..     504     522 .. 0.92\n",
            "   3 !   16.8   5.3   8.2e-06    0.0049       1      27 []     559     585 ..     559     585 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.9 bits;  conditional E-value: 0.00013\n",
            "                     TSSTCBSHHHHTS--TTG.GG-SSBSCT CS\n",
            "         zf-CCCH   1 yktelCkffmrtGyCkyG.drCkFaHgq 27 \n",
            "                     +kte+C+++m+ G C+    +C+FaHg+\n",
            "  XP_002905007.1 455 WKTEICMYYMQ-GKCNKTrRTCSFAHGE 481\n",
            "                     8**********.***7520369999985 PP\n",
            "\n",
            "  == domain 2  score: 6.2 bits;  conditional E-value: 0.017\n",
            "                     SSTCBSHHHHTS--T CS\n",
            "         zf-CCCH   2 ktelCkffmrtGyCk 16 \n",
            "                     kt+lC+ +   G+C+\n",
            "  XP_002905007.1 505 KTRLCQAYEN-GTCP 518\n",
            "                     9*********.***9 PP\n",
            "\n",
            "  == domain 3  score: 16.8 bits;  conditional E-value: 8.2e-06\n",
            "                     TSSTCBSHHHHTS--TTG.GG-SSBSCT CS\n",
            "         zf-CCCH   1 yktelCkffmrtGyCkyG.drCkFaHgq 27 \n",
            "                     yktelC +f++ G C+y  + C+FaHgq\n",
            "  XP_002905007.1 559 YKTELCYYFLK-GNCNYTkEECRFAHGQ 585\n",
            "                     9**********.*****7468*****97 PP\n",
            "\n",
            ">> XP_002905360.1  uncharacterized protein PITG_04608 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   25.8   0.6   1.2e-08   7.3e-06       1      26 [.     241     265 ..     241     266 .. 0.89\n",
            "   2 ?   -1.9   0.6       5.6   3.3e+03      18      24 ..     340     346 ..     336     348 .. 0.64\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 25.8 bits;  conditional E-value: 1.2e-08\n",
            "                     TSSTCBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH   1 yktelCkffmrtGyCkyGdrCkFaHg 26 \n",
            "                     +k +lC ff+  G C  G+rCkF+H+\n",
            "  XP_002905360.1 241 PKSVLCAFFKA-GVCTKGNRCKFSHD 265\n",
            "                     5899***7777.*************7 PP\n",
            "\n",
            "  == domain 2  score: -1.9 bits;  conditional E-value: 5.6\n",
            "                     GGG-SSB CS\n",
            "         zf-CCCH  18 GdrCkFa 24 \n",
            "                     G +Ck+ \n",
            "  XP_002905360.1 340 GTSCKYR 346\n",
            "                     4567775 PP\n",
            "\n",
            ">> XP_002907667.1  uncharacterized protein PITG_02775 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   16.3   2.5   1.2e-05    0.0069       4      21 ..     385     401 ..     383     403 .. 0.96\n",
            "   2 !   13.8   2.5   6.9e-05     0.041       2      23 ..     431     451 ..     430     452 .. 0.96\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 16.3 bits;  conditional E-value: 1.2e-05\n",
            "                     TCBSHHHHTS--TTGGG- CS\n",
            "         zf-CCCH   4 elCkffmrtGyCkyGdrC 21 \n",
            "                     elC++f+  G C +GdrC\n",
            "  XP_002907667.1 385 ELCRNFLA-GRCTFGDRC 401\n",
            "                     79******.********* PP\n",
            "\n",
            "  == domain 2  score: 13.8 bits;  conditional E-value: 6.9e-05\n",
            "                     SSTCBSHHHHTS--TTGGG-SS CS\n",
            "         zf-CCCH   2 ktelCkffmrtGyCkyGdrCkF 23 \n",
            "                     kt+lC +f   G C++Gd+C++\n",
            "  XP_002907667.1 431 KTVLCNNFPL-GMCRFGDKCRY 451\n",
            "                     9*********.**********8 PP\n",
            "\n",
            ">> XP_002907473.1  tRNA-dihydrouridine synthase 3 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.9   0.9   3.7e-06    0.0022      14      26 ..     110     122 ..     101     123 .. 0.83\n",
            "   2 !   11.2   2.1   0.00047      0.28       5      24 ..     137     156 ..     135     156 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.9 bits;  conditional E-value: 3.7e-06\n",
            "                     --TTGGG-SSBSC CS\n",
            "         zf-CCCH  14 yCkyGdrCkFaHg 26 \n",
            "                      Ck+Gd+CkF+H+\n",
            "  XP_002907473.1 110 GCKFGDSCKFSHD 122\n",
            "                     6***********6 PP\n",
            "\n",
            "  == domain 2  score: 11.2 bits;  conditional E-value: 0.00047\n",
            "                     CBSHHHHTS--TTGGG-SSB CS\n",
            "         zf-CCCH   5 lCkffmrtGyCkyGdrCkFa 24 \n",
            "                     +C+ f   GyC+yG +C+Fa\n",
            "  XP_002907473.1 137 RCPVFDAVGYCRYGMACRFA 156\n",
            "                     67777777***********8 PP\n",
            "\n",
            ">> XP_002896200.1  uncharacterized protein PITG_19639 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   16.2   2.5   1.2e-05    0.0071       4      21 ..     401     417 ..     399     419 .. 0.96\n",
            "   2 !   13.8   2.5   7.1e-05     0.042       2      23 ..     447     467 ..     446     468 .. 0.96\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 16.2 bits;  conditional E-value: 1.2e-05\n",
            "                     TCBSHHHHTS--TTGGG- CS\n",
            "         zf-CCCH   4 elCkffmrtGyCkyGdrC 21 \n",
            "                     elC++f+  G C +GdrC\n",
            "  XP_002896200.1 401 ELCRNFLA-GRCTFGDRC 417\n",
            "                     79******.********* PP\n",
            "\n",
            "  == domain 2  score: 13.8 bits;  conditional E-value: 7.1e-05\n",
            "                     SSTCBSHHHHTS--TTGGG-SS CS\n",
            "         zf-CCCH   2 ktelCkffmrtGyCkyGdrCkF 23 \n",
            "                     kt+lC +f   G C++Gd+C++\n",
            "  XP_002896200.1 447 KTVLCNNFPL-GMCRFGDKCRY 467\n",
            "                     9*********.**********8 PP\n",
            "\n",
            ">> XP_002997690.1  uncharacterized protein PITG_18056 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   22.3   4.2   1.5e-07   8.6e-05       2      27 .]      48      72 ..      47      72 .. 0.95\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 22.3 bits;  conditional E-value: 1.5e-07\n",
            "                    SSTCBSHHHHTS--TTGGG-SSBSCT CS\n",
            "         zf-CCCH  2 ktelCkffmrtGyCkyGdrCkFaHgq 27\n",
            "                    k +lCk+f + G C++ ++CkFaHg+\n",
            "  XP_002997690.1 48 KSALCKHFDKPGGCPF-PNCKFAHGR 72\n",
            "                    789*************.*******95 PP\n",
            "\n",
            ">> XP_002902613.1  uncharacterized protein PITG_10082 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   14.6   6.6   3.8e-05     0.023       6      25 ..      40      58 ..      36      60 .. 0.93\n",
            "   2 !   21.4   4.7   2.8e-07   0.00017       1      26 [.      95     119 ..      95     120 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 14.6 bits;  conditional E-value: 3.8e-05\n",
            "                    BSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH  6 CkffmrtGyCkyGdrCkFaH 25\n",
            "                    C+ff+r   C+ G++CkF H\n",
            "  XP_002902613.1 40 CHFFAR-AKCRDGEKCKFLH 58\n",
            "                    ******.************* PP\n",
            "\n",
            "  == domain 2  score: 21.4 bits;  conditional E-value: 2.8e-07\n",
            "                     TSSTCBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH   1 yktelCkffmrtGyCkyGdrCkFaHg 26 \n",
            "                     +kt+ Ck+f+    C+ Gd+CkF+H+\n",
            "  XP_002902613.1  95 KKTRKCKYFAW-NKCRNGDKCKFSHD 119\n",
            "                     69*********.*************8 PP\n",
            "\n",
            ">> XP_002908060.1  uncharacterized protein PITG_01390 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   12.6   1.2   0.00016     0.095       5      26 ..     250     269 ..     249     270 .. 0.93\n",
            "   2 !   10.3   0.4   0.00084       0.5       6      26 ..     275     294 ..     273     295 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.6 bits;  conditional E-value: 0.00016\n",
            "                     CBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH   5 lCkffmrtGyCkyGdrCkFaHg 26 \n",
            "                      C+++++ G+C++ + C F H+\n",
            "  XP_002908060.1 250 VCRHYLQ-GECRR-ADCMFLHD 269\n",
            "                     6******.*****.*******7 PP\n",
            "\n",
            "  == domain 2  score: 10.3 bits;  conditional E-value: 0.00084\n",
            "                     BSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH   6 CkffmrtGyCkyGdrCkFaHg 26 \n",
            "                     C+f++r G+C   d C FaH+\n",
            "  XP_002908060.1 275 CRFWLR-GTCLQADHCVFAHD 294\n",
            "                     ******.*************7 PP\n",
            "\n",
            ">> XP_002906927.1  uncharacterized protein PITG_03888 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.8   0.9       2.6   1.6e+03       6      15 ..       2      10 ..       2      15 .. 0.82\n",
            "   2 !    9.3   1.2    0.0018       1.1       2      26 ..      25      47 ..      24      48 .. 0.89\n",
            "   3 !   19.9   1.0   8.4e-07    0.0005       4      22 ..      54      71 ..      53      71 .. 0.95\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.8 bits;  conditional E-value: 2.6\n",
            "                    BSHHHHTS-- CS\n",
            "         zf-CCCH  6 CkffmrtGyC 15\n",
            "                    C+ f++  +C\n",
            "  XP_002906927.1  2 CRKFLK-NEC 10\n",
            "                    999*99.777 PP\n",
            "\n",
            "  == domain 2  score: 9.3 bits;  conditional E-value: 0.0018\n",
            "                    SSTCBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH  2 ktelCkffmrtGyCkyGdrCkFaHg 26\n",
            "                    k++ Ck f+r G+C + + Ck+ H+\n",
            "  XP_002906927.1 25 KVPDCKMFLR-GACTR-EGCKYRHV 47\n",
            "                    899*******.*****.88999886 PP\n",
            "\n",
            "  == domain 3  score: 19.9 bits;  conditional E-value: 8.4e-07\n",
            "                    TCBSHHHHTS--TTGGG-S CS\n",
            "         zf-CCCH  4 elCkffmrtGyCkyGdrCk 22\n",
            "                    elC  fm+ GyC+ G++C+\n",
            "  XP_002906927.1 54 ELCEQFMK-GYCPKGEACP 71\n",
            "                    79******.*********5 PP\n",
            "\n",
            ">> XP_002907409.1  uncharacterized protein PITG_02485 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.1   1.9   6.5e-06    0.0039       4      26 ..     135     157 ..     132     158 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.1 bits;  conditional E-value: 6.5e-06\n",
            "                     TCBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH   4 elCkffmrtGyCkyGdrCkFaHg 26 \n",
            "                     + Ckf+  + +C++Gd+C+F H+\n",
            "  XP_002907409.1 135 AHCKFWINSKTCQHGDNCEFFHV 157\n",
            "                     66******************997 PP\n",
            "\n",
            ">> XP_002897011.1  uncharacterized protein PITG_17110 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.7   7.2   1.8e-05     0.011       2      26 ..     430     454 ..     429     455 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.7 bits;  conditional E-value: 1.8e-05\n",
            "                     SSTCBSHHHHTS--TTGGG-SSBSC CS\n",
            "         zf-CCCH   2 ktelCkffmrtGyCkyGdrCkFaHg 26 \n",
            "                     +t  C +f+  G C++G +C+F+H+\n",
            "  XP_002897011.1 430 GTGVCYDFQNKGVCQRGRFCHFSHC 454\n",
            "                     7889********************7 PP\n",
            "\n",
            ">> XP_002904939.1  uncharacterized protein PITG_06966 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    3.7   4.1       0.1        60       5      25 ..       6      26 ..       3      27 .. 0.90\n",
            "   2 !   16.0   2.6   1.4e-05    0.0082       3      25 ..      56      78 ..      54      79 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 3.7 bits;  conditional E-value: 0.1\n",
            "                    CBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH  5 lCkffmrtGyCkyGdrCkFaH 25\n",
            "                     C+ff+    C +G++C F H\n",
            "  XP_002904939.1  6 ACRFFQSPQGCFRGASCTFRH 26\n",
            "                    699999999**********88 PP\n",
            "\n",
            "  == domain 2  score: 16.0 bits;  conditional E-value: 1.4e-05\n",
            "                    STCBSHHHHTS--TTGGG-SSBS CS\n",
            "         zf-CCCH  3 telCkffmrtGyCkyGdrCkFaH 25\n",
            "                     ++C+ff     C+ G++C+F H\n",
            "  XP_002904939.1 56 PRPCHFFTSENGCRNGSNCPFVH 78\n",
            "                    589******************** PP\n",
            "\n",
            ">> XP_002909043.1  uncharacterized protein PITG_00444 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.3   4.3   9.7e-05     0.057       3      27 .]     699     723 ..     697     723 .. 0.85\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.3 bits;  conditional E-value: 9.7e-05\n",
            "                     STCBSHHHHTS--TTGGG-SSBSCT CS\n",
            "         zf-CCCH   3 telCkffmrtGyCkyGdrCkFaHgq 27 \n",
            "                     t +C+ff+    Ck+G++C++aH+q\n",
            "  XP_002909043.1 699 TLPCRFFASPMGCKHGGNCHYAHVQ 723\n",
            "                     668998888779***********86 PP\n",
            "\n",
            ">> XP_002908931.1  uncharacterized protein PITG_00313 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.1   0.1       1.6   9.2e+02       3      10 ..     126     133 ..     125     135 .. 0.80\n",
            "   2 ?   12.2   1.1   0.00022      0.13       2      26 ..     353     380 ..     352     381 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.1 bits;  conditional E-value: 1.6\n",
            "                     STCBSHHH CS\n",
            "         zf-CCCH   3 telCkffm 10 \n",
            "                     + +C+ff+\n",
            "  XP_002908931.1 126 EQPCRFFS 133\n",
            "                     579***97 PP\n",
            "\n",
            "  == domain 2  score: 12.2 bits;  conditional E-value: 0.00022\n",
            "                     SSTCBSHHHHTS--TTG...GG-SSBSC CS\n",
            "         zf-CCCH   2 ktelCkffmrtGyCkyG...drCkFaHg 26 \n",
            "                      +++C+ f+++G+C      +rC++aH+\n",
            "  XP_002908931.1 353 ERPICPSFRQRGICINVqklGRCRYAHP 380\n",
            "                     689***********99977789*****8 PP\n",
            "\n",
            ">> XP_002906972.1  uncharacterized protein PITG_03939 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.9   0.0       2.7   1.6e+03       3      11 ..     177     185 ..     176     188 .. 0.81\n",
            "   2 ?    8.4   1.4    0.0034         2      15      27 .]     203     215 ..     202     215 .. 0.95\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.9 bits;  conditional E-value: 2.7\n",
            "                     STCBSHHHH CS\n",
            "         zf-CCCH   3 telCkffmr 11 \n",
            "                     t+lCk + +\n",
            "  XP_002906972.1 177 TVLCKSWVK 185\n",
            "                     89***9976 PP\n",
            "\n",
            "  == domain 2  score: 8.4 bits;  conditional E-value: 0.0034\n",
            "                     -TTGGG-SSBSCT CS\n",
            "         zf-CCCH  15 CkyGdrCkFaHgq 27 \n",
            "                     C++G  C FaHg+\n",
            "  XP_002906972.1 203 CPRGRDCDFAHGE 215\n",
            "                     ***********96 PP\n",
            "\n",
            ">> XP_002905649.1  uncharacterized protein PITG_04956 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.0   4.5     0.081        48       2      26 ..     605     629 ..     604     630 .. 0.82\n",
            "   2 ?    6.7   1.2     0.012         7      13      26 ..     642     655 ..     640     656 .. 0.91\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.0 bits;  conditional E-value: 0.081\n",
            "                     SSTCBSHHHHTS--TTGGG.-SSBSC CS\n",
            "         zf-CCCH   2 ktelCkffmrtGyCkyGdr.CkFaHg 26 \n",
            "                     kt lCk f+r G+C++ d+ Ck++H+\n",
            "  XP_002905649.1 605 KTMLCKWFSR-GHCRFDDAhCKYSHD 629\n",
            "                     6779**8999.*****7645999997 PP\n",
            "\n",
            "  == domain 2  score: 6.7 bits;  conditional E-value: 0.012\n",
            "                     S--TTGGG-SSBSC CS\n",
            "         zf-CCCH  13 GyCkyGdrCkFaHg 26 \n",
            "                     G+C+ G  C+F+H+\n",
            "  XP_002905649.1 642 GSCSKGMYCPFSHD 655\n",
            "                     9************7 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (27 nodes)\n",
            "Target sequences:                      17797  (7477560 residues searched)\n",
            "Passed MSV filter:                       536  (0.0301174); expected 355.9 (0.02)\n",
            "Passed bias filter:                      354  (0.019891); expected 355.9 (0.02)\n",
            "Passed Vit filter:                        55  (0.00309041); expected 17.8 (0.001)\n",
            "Passed Fwd filter:                        34  (0.00191043); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              17797  [actual number of targets]\n",
            "Domain search space  (domZ):              30  [number of targets reported over threshold]\n",
            "# CPU time: 0.15u 0.01s 00:00:00.16 Elapsed: 00:00:00.28\n",
            "# Mc/sec: 699.97\n",
            "//\n",
            "Query:       Wtap  [M=155]\n",
            "Accession:   PF17098.8\n",
            "Description: WTAP/Mum2p family\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.016   15.2   0.3      0.016   15.2   0.3    2.3  2  XP_002900699.1  sporangia induced conserved hypothetical prot\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_002900699.1  sporangia induced conserved hypothetical protein [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    2.4   0.8    0.0079   1.4e+02      15      75 ..      71     130 ..      62     132 .. 0.72\n",
            "   2 ?   15.2   0.3   9.1e-07     0.016     109     155 .]     126     172 ..     121     172 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 2.4 bits;  conditional E-value: 0.0079\n",
            "            Wtap  15 akeqeiaelkaslrdlrqalrpessqlrqlllDPavnlefkkLkaeleeaekklrqaqeeL 75 \n",
            "                     + e  ++  ++ +++l + ++ ++ +l  + ++       ++L++   +++kkl++a ee+\n",
            "  XP_002900699.1  71 SLECKVQSHEDTIQKLMSIHQADK-TLLLTKIEEITDRTTEELHKLRIDYQKKLNTASEEI 130\n",
            "                     556667777777888888877765.777777777777777888888788888888877765 PP\n",
            "\n",
            "  == domain 2  score: 15.2 bits;  conditional E-value: 9.1e-07\n",
            "            Wtap 109 eegkvaqLeselalqkelveelkkaleeleevleeldeeleglqskl 155\n",
            "                     +++++ +L+  l++q+ ++  l+++++ l++++ ++d ++ +l++++\n",
            "  XP_002900699.1 126 ASEEIDRLNKCLNVQRGHITTLQEQCQSLHKHVVQVDDDVTKLATEV 172\n",
            "                     57899************************************999875 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (155 nodes)\n",
            "Target sequences:                      17797  (7477560 residues searched)\n",
            "Passed MSV filter:                      1846  (0.103725); expected 355.9 (0.02)\n",
            "Passed bias filter:                      537  (0.0301736); expected 355.9 (0.02)\n",
            "Passed Vit filter:                        64  (0.00359611); expected 17.8 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000112378); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              17797  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.27u 0.01s 00:00:00.28 Elapsed: 00:00:00.29\n",
            "# Mc/sec: 3983.88\n",
            "//\n",
            "Query:       VIR_N  [M=265]\n",
            "Accession:   PF15912.8\n",
            "Description: Virilizer, N-terminal\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "     0.0062   16.2   4.6     0.0071   16.0   4.6    1.1  1  XP_002905433.1  uncharacterized protein PITG_04699 [Phytophth\n",
            "  ------ inclusion threshold ------\n",
            "       0.13   11.9   8.4       0.22   11.1   8.4    1.3  1  XP_002904175.1  uncharacterized protein PITG_07971 [Phytophth\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_002905433.1  uncharacterized protein PITG_04699 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   16.0   4.6     8e-07    0.0071     135     218 ..       7      87 ..       1     131 [. 0.49\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 16.0 bits;  conditional E-value: 8e-07\n",
            "           VIR_N 135 pPppPapPvkqekaelekedqynaspprpeprePrtpelipaytgnvpgtpPpdddeeedaavpetveeekeensekredylee 218\n",
            "                     pPppP+pP++q+  + +k  + n + p+p++r+   + +++ ++++  ++     + e  a  ++t++e++e ++ +r++  + \n",
            "  XP_002905433.1   7 PPPPPPPPPPQDDLKRKKPTLNNPNQPKPRKRKDFRA-IMALFHDEGKDMAK--VKREWAALWQTTLTESQEDHKRRRQQEQHG 87 \n",
            "                     3444444434444444555555555555555555555.55566665544444..566665566666664444443333333222 PP\n",
            "\n",
            ">> XP_002904175.1  uncharacterized protein PITG_07971 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.1   8.4   2.5e-05      0.22     135     259 ..     253     367 ..     225     372 .. 0.66\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.1 bits;  conditional E-value: 2.5e-05\n",
            "           VIR_N 135 pPppPapPvkqekaelekedqynaspprpeprePrtpelipaytgnvpgtpPpdddeeedaavpetveeekeensekredyleeesP.eertslk 228\n",
            "                      P+ Pa+P    +a     d +na  p      P +p   +  ++ + ++p +ddd+ + +  p+t+  e e +s+  +d ++  +P     s+k\n",
            "  XP_002904175.1 253 APATPAQP---ANA---TSDFLNAMDPLATRSAPSQP---SPSKTVLRDPPKFDDDNGQ-TVGPVTMA-EMEAHSKSTSDGSNVYNPnLVDKSTK 336\n",
            "                     34444444...333...55666666666666666666...2235556777777888777.66677766.56666666666666677742344677 PP\n",
            "\n",
            "           VIR_N 229 aeekyseaereeresseeierdeeesdvrer 259\n",
            "                     +++  +  e+ er+ ++++er +++   re+\n",
            "  XP_002904175.1 337 SDDVRRAMEERERQVQRDVERARDDLRKREE 367\n",
            "                     888888888999999********98666655 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (265 nodes)\n",
            "Target sequences:                      17797  (7477560 residues searched)\n",
            "Passed MSV filter:                      1866  (0.104849); expected 355.9 (0.02)\n",
            "Passed bias filter:                      962  (0.0540541); expected 355.9 (0.02)\n",
            "Passed Vit filter:                       200  (0.0112378); expected 17.8 (0.001)\n",
            "Passed Fwd filter:                        60  (0.00337135); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              17797  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.92u 0.01s 00:00:00.93 Elapsed: 00:00:00.74\n",
            "# Mc/sec: 2658.86\n",
            "//\n",
            "Query:       TruB_N  [M=149]\n",
            "Accession:   PF01509.21\n",
            "Description: TruB family pseudouridylate synthase (N terminal domain)\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    5.4e-44  150.2   0.0    7.4e-44  149.7   0.0    1.2  1  XP_002997322.1  tRNA pseudouridine synthase B, putative [Phyt\n",
            "    7.2e-24   84.9   0.8    7.9e-21   75.0   0.8    3.5  1  XP_002998420.1  H/ACA ribonucleoprotein complex subunit 4 [Ph\n",
            "  ------ inclusion threshold ------\n",
            "        0.6   10.5   3.1        8.8    6.7   0.0    2.6  2  XP_002901946.1  uncharacterized protein PITG_11183 [Phytophth\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_002997322.1  tRNA pseudouridine synthase B, putative [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  149.7   0.0   1.2e-47   7.4e-44       1     135 [.      23     156 ..      23     166 .. 0.96\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 149.7 bits;  conditional E-value: 1.2e-47\n",
            "                     HHHTT-S-EEESS---TT-EEEEEEEEGGGGGGHHHCTTS-EEEEEEEEETEEETTSBTTS-EEE-.--TT--HHHHHHHHHHC-EEE------- CS\n",
            "          TruB_N   1 krllkakkvGhtGtLDplatGvLvvavgeaTklleylleadKeYeatirlgaetdteDaegeileeekveklteekleevlkkftGeieqvppmy 95 \n",
            "                     +r+l++++vGh+GtLDp+atGvL++avg+aT++l+yl++  KeY+  irlg++td++D +g++ ++  ++ ++ee+++ +l++f Gei+qvpp +\n",
            "  XP_002997322.1  23 RRVLDTRQVGHGGTLDPMATGVLMLAVGRATRFLQYLTTG-KEYRGAIRLGITTDSDDITGDVRTQLPAPWISEEMVNMTLQRFIGEIDQVPPRV 116\n",
            "                     899**********************************987.**********************999679************************** PP\n",
            "\n",
            "                     --------------------------EEEEEEEEEEEEEE CS\n",
            "          TruB_N  96 SAvkvnGkrlyelaregeeverkkrkvtikelelleveep 135\n",
            "                     SA+k+nG rly+lar+ +++++k+r+v ik++++ ++++ \n",
            "  XP_002997322.1 117 SAIKKNGVRLYKLARAKKDFDVKPRRVCIKKIDVQSFTQG 156\n",
            "                     **********************************999765 PP\n",
            "\n",
            ">> XP_002998420.1  H/ACA ribonucleoprotein complex subunit 4 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   75.0   0.8   1.3e-24   7.9e-21       1     149 []      98     214 ..      98     214 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 75.0 bits;  conditional E-value: 1.3e-24\n",
            "                     HHHTT-S-EEESS---TT-EEEEEEEEGGGGGGHHHCTTS-EEEEEEEEETEEETTSBTTS-EEE-.--TT--HHHHHHHHHHC-EEE------- CS\n",
            "          TruB_N   1 krllkakkvGhtGtLDplatGvLvvavgeaTklleylleadKeYeatirlgaetdteDaegeileeekveklteekleevlkkftGeieqvppmy 95 \n",
            "                     +r+l+++k Gh+GtLDp++tG+L+v+v++aT+l++  + a KeY+a++rl+ + +++                  kl ++++++tG++ q+pp +\n",
            "  XP_002998420.1  98 RRILRVDKTGHSGTLDPKVTGCLIVCVDRATRLVKAQQGAGKEYVAVVRLHSAIESQV-----------------KLARAIETLTGALFQRPPLI 175\n",
            "                     89*************************************************9987764.................58899*************** PP\n",
            "\n",
            "                     --------------------------EEEEEEEEEEEEEE..EEEEEEEESTTTTH CS\n",
            "          TruB_N  96 SAvkvnGkrlyelaregeeverkkrkvtikelelleveep..evelevecskGtYi 149\n",
            "                     SAvk                 r+ r  ti+e +l+e++e+     ++v+c++GtYi\n",
            "  XP_002998420.1 176 SAVK-----------------RQLRIRTIYESKLIEYDEDrhLGIFHVSCEAGTYI 214\n",
            "                     ****.................999*************988756789*********8 PP\n",
            "\n",
            ">> XP_002901946.1  uncharacterized protein PITG_11183 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.7   0.0    0.0015       8.8      12      30 ..     266     284 ..     260     295 .. 0.81\n",
            "   2 ?    2.9   1.2     0.021   1.2e+02      56     112 ..     350     406 ..     320     448 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.7 bits;  conditional E-value: 0.0015\n",
            "                     SS---TT-EEEEEEEEGGG CS\n",
            "          TruB_N  12 tGtLDplatGvLvvavgea 30 \n",
            "                     t tLDpla+GvL  a+ +a\n",
            "  XP_002901946.1 266 TETLDPLANGVLNSAIASA 284\n",
            "                     77**********8777665 PP\n",
            "\n",
            "  == domain 2  score: 2.9 bits;  conditional E-value: 0.021\n",
            "                     TSBTTS-EEE-.--TT--HHHHHHHHHHC-EEE------------------------ CS\n",
            "          TruB_N  56 teDaegeileeekveklteekleevlkkftGeieqvppmySAvkvnGkrlyelareg 112\n",
            "                     +  ++ +++++   ++ + +++ +vl++ + +++ +p ++S +k +  ++ +   ++\n",
            "  XP_002901946.1 350 QFLQQQQTIASLIRQQHDLKQIINVLQEQQQQLMTIPTQISELKRQRANINNGETRD 406\n",
            "                     444445555553456777888999999999999999*99999997765555444444 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (149 nodes)\n",
            "Target sequences:                      17797  (7477560 residues searched)\n",
            "Passed MSV filter:                       559  (0.0314098); expected 355.9 (0.02)\n",
            "Passed bias filter:                      395  (0.0221948); expected 355.9 (0.02)\n",
            "Passed Vit filter:                        37  (0.002079); expected 17.8 (0.001)\n",
            "Passed Fwd filter:                         3  (0.000168568); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              17797  [actual number of targets]\n",
            "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
            "# CPU time: 0.18u 0.00s 00:00:00.18 Elapsed: 00:00:00.09\n",
            "# Mc/sec: 11346.76\n",
            "//\n",
            "Query:       THRAP3_BCLAF1  [M=662]\n",
            "Accession:   PF15440.9\n",
            "Description: THRAP3/BCLAF1 family\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "     0.0078   15.3  27.5      0.008   15.2  27.5    1.1  1  XP_002900404.1  uncharacterized protein PITG_12525 [Phytophth\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_002900404.1  uncharacterized protein PITG_12525 [Phytophthora infestans T30-4]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.2  27.5   4.5e-07     0.008       2      87 ..       3      91 ..       2     162 .. 0.57\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.2 bits;  conditional E-value: 4.5e-07\n",
            "   THRAP3_BCLAF1  2 RSRSRS.P....KrRSlSP.rsRsrsrysdRsssgrSrs.....sssrsssnhskdpkrskrrrskekkagqkdkrisqaanehnrsyeerspdaks 87\n",
            "                    RSRSRS P    +rR +SP rsRs+ r+  R s +rSrs     + srs + h k+ +++        k  +k+k+   ++n+h  ++e++ +++++\n",
            "  XP_002900404.1  3 RSRSRSvPrdrpRRRHRSPsRSRSHGRRLSRRS-KRSRSpvdgrRRSRSRKRHHKKHRKH-------LKRAKKEKKKRHNKNKHRENQEKELEETAT 91\n",
            "                    9*****845555777777724555555544444.33555444444444444444444433.......333333333333333333333333331111 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (662 nodes)\n",
            "Target sequences:                      17797  (7477560 residues searched)\n",
            "Passed MSV filter:                      3268  (0.183626); expected 355.9 (0.02)\n",
            "Passed bias filter:                      683  (0.0383773); expected 355.9 (0.02)\n",
            "Passed Vit filter:                       160  (0.00899028); expected 17.8 (0.001)\n",
            "Passed Fwd filter:                         7  (0.000393325); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              17797  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.98u 0.02s 00:00:01.00 Elapsed: 00:00:00.52\n",
            "# Mc/sec: 9360.34\n",
            "//\n",
            "Query:       SRRM_C  [M=67]\n",
            "Accession:   PF15230.9\n",
            "Description: Serine/arginine repetitive matrix protein C-terminus\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (67 nodes)\n",
            "Target sequences:                      17797  (7477560 residues searched)\n",
            "Passed MSV filter:                      2120  (0.119121); expected 355.9 (0.02)\n",
            "Passed bias filter:                      595  (0.0334326); expected 355.9 (0.02)\n",
            "Passed Vit filter:                        69  (0.00387706); expected 17.8 (0.001)\n",
            "Passed Fwd filter:                         1  (5.61892e-05); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              17797  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.19u 0.01s 00:00:00.20 Elapsed: 00:00:00.10\n",
            "# Mc/sec: 4684.56\n",
            "//\n",
            "[ok]\n"
          ]
        }
      ]
    }
  ]
}